From 21fb1aca46d2ca90ef2f350e10b4b6f37f976c6c Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <michal.hrusecky@turris.com>
Date: Mon, 29 Sep 2025 09:32:52 +0200
Subject: [PATCH] wayland: Add libiconv dependency

---
 frameworks/wayland/Makefile | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/frameworks/wayland/Makefile b/frameworks/wayland/Makefile
index 192fc9e..f4ea926 100644
--- a/frameworks/wayland/Makefile
+++ b/frameworks/wayland/Makefile
@@ -26,7 +26,7 @@ define Package/libwayland
   SUBMENU:=Frameworks and Toolkits
   TITLE:=wayland
   URL:=https://wayland.freedesktop.org/
-  DEPENDS:=+libexpat +libffi
+  DEPENDS:=+libexpat +libffi +libiconv-full
   USERID:=:video=40
 endef
 
@@ -38,7 +38,7 @@ define Package/wayland-scanner
   SUBMENU:=Frameworks and Toolkits
   TITLE:=wayland
   URL:=https://wayland.freedesktop.org/
-  DEPENDS:=+libxml2 +libexpat
+  DEPENDS:=+libxml2 +libexpat +libiconv-full
 endef
 
 define Package/wayland-scanner/description
@@ -51,6 +51,7 @@ MESON_HOST_ARGS += \
 	-Ddtd_validation=true
 
 HOST_LDFLAGS += $(STAGING_DIR_HOST)/lib/libz.a -lm
+TARGET_LDFLAGS += -L$(STAGING_DIR)/usr/lib/libiconv-full/lib -liconv
 
 MESON_ARGS += \
 	-Dscanner=true \
-- 
2.51.0

