From 2c921152b155328f184f5f18e498fdda009e98a7 Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <michal.hrusecky@turris.com>
Date: Mon, 29 Sep 2025 13:20:45 +0200
Subject: [PATCH] libxkbcommon: Add libiconv dependency

---
 libs/libxkbcommon/Makefile | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/libs/libxkbcommon/Makefile b/libs/libxkbcommon/Makefile
index 58277e9..95495dc 100644
--- a/libs/libxkbcommon/Makefile
+++ b/libs/libxkbcommon/Makefile
@@ -29,12 +29,14 @@ define Package/libxkbcommon
   CATEGORY:=Libraries
   TITLE:=libxkbcommon
   URL:=https://xkbcommon.org/
-  DEPENDS:=+libwayland +libxml2 +wayland-protocols
+  DEPENDS:=+libwayland +libxml2 +wayland-protocols +libiconv-full
 endef
 
 define Package/libxkbcommon/description
 endef
 
+TARGET_LDFLAGS += -L$(STAGING_DIR)/usr/lib/libiconv-full/lib -liconv
+
 define Build/InstallDev
 	$(INSTALL_DIR) $(1)/usr/include
 	$(CP) $(PKG_INSTALL_DIR)/usr/include/* $(1)/usr/include/
-- 
2.51.0

