From 5c8e25dd895f7cd5f74f0e4fe917445da70e7be5 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Fri, 17 Sep 2021 17:08:09 +0200
Subject: [PATCH] foris: remove Foris in favor of reForis

---
 foris.lua.m4 | 25 -------------------------
 1 file changed, 25 deletions(-)

diff --git a/foris.lua.m4 b/foris.lua.m4
index 4c9068a..9317eb2 100644
--- a/foris.lua.m4
+++ b/foris.lua.m4
@@ -1,15 +1,6 @@
 include(utils.m4)dnl
 _FEATURE_GUARD_
 
-local foris_plugins = {
-	["diagnostics"] = "turris-diagnostics",
-	["netmetr"] = "netmetr",
-	["openvpn"] = "openvpn",
-	["pakon"] = "pakon",
-	["storage"] = false,
-	["subordinates"] = "turris-netboot-tools",
-}
-
 local reforis_plugins = {
 	["data-collection"] = "sentinel-proxy",
 	["haas"] = "haas-proxy",
@@ -25,23 +16,8 @@ local reforis_plugins = {
 
 ----------------------------------------------------------------------------------
 
-Install("foris", "foris-storage-plugin", { priority = 40 })
 Install("reforis", "reforis-storage-plugin", { priority = 40 })
 
-for plugin, condition in pairs(foris_plugins) do
-	local fplugin = "foris-" .. plugin .. "-plugin"
-	if condition ~= false then
-		Install(fplugin, { priority = 40, condition = condition })
-	end
-	for _, lang in pairs(l10n or {}) do
-		Install(fplugin .. "-l10n-" .. lang, {
-			priority = 10,
-			optional = true,
-			condition = fplugin
-		})
-	end
-end
-
 for plugin, condition in pairs(reforis_plugins) do
 	local refplugin = "reforis-" .. plugin .. "-plugin"
 	if condition ~= false then
@@ -57,7 +33,6 @@ for plugin, condition in pairs(reforis_plugins) do
 end
 
 if for_l10n then
-	for_l10n("foris-l10n-")
 	for_l10n("reforis-l10n-")
 	for_l10n('pkglists-l10n-')
 end
-- 
2.30.2

