From 122455937b6a1c829e41556cf08f6cffee95c049 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Fri, 17 Sep 2021 17:08:09 +0200
Subject: [PATCH] foris: remove Foris in favor of reForis

---
 foris.lua.m4 | 24 ------------------------
 1 file changed, 24 deletions(-)

diff --git a/foris.lua.m4 b/foris.lua.m4
index d62572c..c2921be 100644
--- a/foris.lua.m4
+++ b/foris.lua.m4
@@ -1,15 +1,6 @@
 include(utils.m4)dnl
 _FEATURE_GUARD_
 
-local foris_plugins = {
-	["diagnostics"] = {"foris", "turris-diagnostics"},
-	["netmetr"] = {"foris", "netmetr"},
-	["openvpn"] = {"foris", "openvpn"},
-	["pakon"] = "pakon",
-	["storage"] = "foris",
-	["subordinates"] = {"foris", "turris-netboot-tools"},
-}
-
 local reforis_plugins = {
 	["data-collection"] = "sentinel-proxy",
 	["haas"] = "haas-proxy",
@@ -28,20 +19,6 @@ local reforis_plugins = {
 Install("lighttpd-https-cert", { priority = 40 })
 Install("reforis", "reforis-storage-plugin", { priority = 40 })
 
-for plugin, condition in pairs(foris_plugins) do
-	local fplugin = "foris-" .. plugin .. "-plugin"
-	if condition ~= false then
-		Install(fplugin, { priority = 40, condition = condition })
-	end
-	for _, lang in pairs(l10n or {}) do
-		Install(fplugin .. "-l10n-" .. lang, {
-			priority = 10,
-			optional = true,
-			condition = fplugin
-		})
-	end
-end
-
 for plugin, condition in pairs(reforis_plugins) do
 	local refplugin = "reforis-" .. plugin .. "-plugin"
 	if condition ~= false then
@@ -56,7 +33,6 @@ for plugin, condition in pairs(reforis_plugins) do
 	end
 end
 
-for_l10n("foris-l10n-", "foris")
 for_l10n("reforis-l10n-")
 for_l10n('pkglists-l10n-')
 
-- 
2.33.0

