From 8941d3c29ad9947d2fb2a27c60ae1c9fcb53cecd Mon Sep 17 00:00:00 2001
From: Richard Muzik <richard.muzik@nic.cz>
Date: Mon, 22 Apr 2024 11:14:44 +0200
Subject: [PATCH] Fix download of node v16.x

---
 node/Makefile | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/node/Makefile b/node/Makefile
index 0ac0bbb4..5ec686a4 100644
--- a/node/Makefile
+++ b/node/Makefile
@@ -30,7 +30,11 @@ NODE_MODULE_VERSION:=108
 endif
 endif
 
+ifeq ($(PKG_VERSION),v16.20.2)
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
+else
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+endif
 PKG_SOURCE_URL:=https://nodejs.org/dist/$(PKG_VERSION)
 
 PKG_MAINTAINER:=Hirokazu MORIKAWA <morikw2@gmail.com>
-- 
2.44.0

