From 71b5cc7d6e6754858bfcb1c1dd862ad51ab15558 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <josef.schlehofer@nic.cz>
Date: Wed, 27 Mar 2019 02:12:12 +0100
Subject: [PATCH] asterisk16: add liburiparser as dependency

---
 net/asterisk-16.x/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/asterisk-16.x/Makefile b/net/asterisk-16.x/Makefile
index 6c1d2cb..622ab4e 100644
--- a/net/asterisk-16.x/Makefile
+++ b/net/asterisk-16.x/Makefile
@@ -443,7 +443,7 @@ define Package/$(PKG_NAME)
 $(call Package/$(PKG_NAME)/Default)
   TITLE:=Complete open source PBX, v$(PKG_VERSION)
   MENU:=1
-  DEPENDS:=$(CXX_DEPENDS) +jansson +libcap +libedit +libopenssl +libsqlite3 +libuuid +libxml2 +zlib
+  DEPENDS:=$(CXX_DEPENDS) +jansson +libcap +libedit +libopenssl +libsqlite3 +libuuid +libxml2 +liburiparser +zlib
   USERID:=asterisk=385:asterisk=385
 endef
 
-- 
2.26.0

