From 8b7850a187064b57bdfdbaa750da404798522109 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <josef.schlehofer@nic.cz>
Date: Wed, 27 Mar 2019 02:12:12 +0100
Subject: [PATCH] asterisk16: add liburiparser as dependency

---
 net/asterisk-16.x/Makefile | 2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/net/asterisk-16.x/Makefile b/net/asterisk-16.x/Makefile
index 92f68f7..9d9378d 100644
--- a/net/asterisk-16.x/Makefile
+++ b/net/asterisk-16.x/Makefile
@@ -442,7 +442,7 @@ define Package/$(PKG_NAME)
 $(call Package/$(PKG_NAME)/Default)
   TITLE:=Complete open source PBX, v$(PKG_VERSION)
   MENU:=1
-  DEPENDS:=$(CXX_DEPENDS) +jansson +libcap +libedit +libopenssl +libsqlite3 +libuuid +libxml2 +zlib
+  DEPENDS:=$(CXX_DEPENDS) +jansson +libcap +libedit +libopenssl +libsqlite3 +libuuid +libxml2 +liburiparser +zlib
 endef

 define Package/$(PKG_NAME)/description
--
2.21.0
