From 8f655bef42f0d579870f49db92fa41a9f28b7e94 Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <Michal@Hrusecky.net>
Date: Thu, 19 Jul 2018 08:36:14 +0200
Subject: [PATCH] hostpad: Revert patches that might be cause of build failure

---
 package/network/services/hostapd/Makefile | 214 +++++++++++++---------
 1 file changed, 130 insertions(+), 84 deletions(-)

diff --git a/package/network/services/hostapd/Makefile b/package/network/services/hostapd/Makefile
index 2941c92..cf0b005 100644
--- a/package/network/services/hostapd/Makefile
+++ b/package/network/services/hostapd/Makefile
@@ -32,10 +32,22 @@ PKG_CONFIG_DEPENDS:= \
 	CONFIG_DRIVER_11N_SUPPORT \
 	CONFIG_DRIVER_11AC_SUPPORT \
 
+WPAD_PROVIDERS:=wpad-mini wpad wpad-openssl wpad-wolfssl \
+		wpad-mesh-openssl wpad-mesh-wolfssl
+
+SUPPLICANT_ONLY_PROVIDERS:=wpa-supplicant-mini wpa-supplicant-p2p \
+			   wpa-supplicant wpa-supplicant-openssl wpa-supplicant-wolfssl \
+			   wpa-supplicant-mesh-openssl wpa-supplicant-mesh-wolfssl
+
+HOSTAPD_ONLY_PROVIDERS:=hostapd-mini hostapd hostapd-openssl hostapd-wolfssl
+
 EAPOL_TEST_PROVIDERS:=eapol-test eapol-test-openssl eapol-test-wolfssl
 
-SUPPLICANT_PROVIDERS:=
-HOSTAPD_PROVIDERS:=
+SUPPLICANT_PROVIDERS:=$(WPAD_PROVIDERS) $(SUPPLICANT_ONLY_PROVIDERS)
+HOSTAPD_PROVIDERS:=$(WPAD_PROVIDERS) $(HOSTAPD_ONLY_PROVIDERS)
+ANY_SUPPLICANT_PROVIDERS:=$(WPAD_PROVIDERS) $(SUPPLICANT_ONLY_PROVIDERS)
+ANY_HOSTAPD_PROVIDERS:=$(WPAD_PROVIDERS) $(HOSTAPD_ONLY_PROVIDERS)
+ANY_PROVIDERS:=$(WPAD_PROVIDERS) $(SUPPLICANT_ONLY_PROVIDERS) $(HOSTAPD_ONLY_PROVIDERS)
 
 LOCAL_TYPE=$(strip \
 		$(if $(findstring wpad,$(BUILD_VARIANT)),wpad, \
@@ -135,51 +147,52 @@ endif
 
 DRV_DEPENDS:=+PACKAGE_kmod-cfg80211:libnl-tiny
 
-
 define Package/hostapd/Default
   SECTION:=net
   CATEGORY:=Network
   TITLE:=IEEE 802.1x Authenticator
   URL:=http://hostap.epitest.fi/
   DEPENDS:=$(DRV_DEPENDS) +hostapd-common +libubus
-  PROVIDES:=hostapd
-  CONFLICTS:=$(HOSTAPD_PROVIDERS)
-  HOSTAPD_PROVIDERS+=$(1)
 endef
 
 define Package/hostapd
-$(call Package/hostapd/Default,$(1))
+$(call Package/hostapd/Default)
   TITLE+= (full)
   VARIANT:=full-internal
 endef
 
-define Package/hostapd/description
- This package contains a full featured IEEE 802.1x/WPA/EAP/RADIUS
- Authenticator.
-endef
-
 define Package/hostapd-openssl
-$(call Package/hostapd/Default,$(1))
+$(call Package/hostapd/Default)
   TITLE+= (full)
   VARIANT:=full-openssl
   DEPENDS+=+libopenssl
+  CONFLICTS:=$(filter-out hostapd-openssl ,$(HOSTAPD_ONLY_PROVIDERS))
+  PROVIDES:=hostapd
 endef
 
-Package/hostapd-openssl/description = $(Package/hostapd/description)
-
 define Package/hostapd-wolfssl
-$(call Package/hostapd/Default,$(1))
+$(call Package/hostapd/Default)
   TITLE+= (full)
   VARIANT:=full-wolfssl
   DEPENDS+=+libwolfssl
+  CONFLICTS:=$(filter-out hostapd-openssl ,$(filter-out hostapd-wolfssl ,$(HOSTAPD_ONLY_PROVIDERS)))
+  PROVIDES:=hostapd
 endef
 
+define Package/hostapd/description
+ This package contains a full featured IEEE 802.1x/WPA/EAP/RADIUS
+ Authenticator.
+endef
+
+Package/hostapd-openssl/description = $(Package/hostapd/description)
 Package/hostapd-wolfssl/description = $(Package/hostapd/description)
 
 define Package/hostapd-mini
-$(call Package/hostapd/Default,$(1))
+$(call Package/hostapd/Default)
   TITLE+= (WPA-PSK only)
   VARIANT:=mini
+  CONFLICTS:=$(filter-out hostapd-wolfssl ,$(filter-out hostapd-openssl ,$(filter-out hostapd-mini ,$(HOSTAPD_ONLY_PROVIDERS))))
+  PROVIDES:=hostapd
 endef
 
 define Package/hostapd-mini/description
@@ -187,6 +200,17 @@ define Package/hostapd-mini/description
 endef
 
 
+define Package/hostapd-utils
+  $(call Package/hostapd/Default)
+  TITLE+= (utils)
+  DEPENDS:=@$(subst $(space),||,$(foreach pkg,$(ANY_HOSTAPD_PROVIDERS),PACKAGE_$(pkg)))
+endef
+
+define Package/hostapd-utils/description
+ This package contains a command line utility to control the
+ IEEE 802.1x/WPA/EAP/RADIUS Authenticator.
+endef
+
 define Package/wpad/Default
   SECTION:=net
   CATEGORY:=Network
@@ -194,54 +218,79 @@ define Package/wpad/Default
   DEPENDS:=$(DRV_DEPENDS) +hostapd-common +libubus
   URL:=http://hostap.epitest.fi/
   PROVIDES:=hostapd wpa-supplicant
-  CONFLICTS:=$(HOSTAPD_PROVIDERS) $(SUPPLICANT_PROVIDERS)
-  HOSTAPD_PROVIDERS+=$(1)
-  SUPPLICANT_PROVIDERS+=$(1)
 endef
 
 define Package/wpad
-$(call Package/wpad/Default,$(1))
+$(call Package/wpad/Default)
   TITLE+= (full)
   VARIANT:=wpad-full-internal
-endef
-
-define Package/wpad/description
- This package contains a full featured IEEE 802.1x/WPA/EAP/RADIUS
- Authenticator and Supplicant
+  CONFLICTS:=$(filter-out wpad-mesh-wolfssl,\
+             $(filter-out wpad-mesh-openssl ,\
+             $(filter-out wpad-openssl ,\
+             $(filter-out wpad-wolfssl ,\
+             $(filter-out wpad-wolfssl ,\
+             $(filter-out wpad ,\
+             $(ANY_PROVIDERS)\
+             ))))))
 endef
 
 define Package/wpad-openssl
-$(call Package/wpad/Default,$(1))
+$(call Package/wpad/Default)
   TITLE+= (full)
   VARIANT:=wpad-full-openssl
   DEPENDS+=+libopenssl
+  CONFLICTS:=$(filter-out wpad-mesh-wolfssl,\
+             $(filter-out wpad-mesh-openssl ,\
+             $(filter-out wpad-openssl ,\
+             $(ANY_PROVIDERS))))
 endef
 
-Package/wpad-openssl/description = $(Package/wpad/description)
-
 define Package/wpad-wolfssl
-$(call Package/wpad/Default,$(1))
+$(call Package/wpad/Default)
   TITLE+= (full)
   VARIANT:=wpad-full-wolfssl
   DEPENDS+=+libwolfssl
+  CONFLICTS:=$(filter-out wpad-mesh-wolfssl ,\
+             $(filter-out wpad-mesh-openssl ,\
+             $(filter-out wpad-openssl ,\
+             $(filter-out wpad-wolfssl ,\
+             $(ANY_PROVIDERS)))))
 endef
 
+define Package/wpad/description
+ This package contains a full featured IEEE 802.1x/WPA/EAP/RADIUS
+ Authenticator and Supplicant
+endef
+
+Package/wpad-openssl/description = $(Package/wpad/description)
 Package/wpad-wolfssl/description = $(Package/wpad/description)
 
 define Package/wpad-mini
-$(call Package/wpad/Default,$(1))
+$(call Package/wpad/Default)
   TITLE+= (WPA-PSK only)
   VARIANT:=wpad-mini
+  CONFLICTS:=$(SUPPLICANT_ONLY_PROVIDERS)
 endef
 
 define Package/wpad-mini/description
  This package contains a minimal IEEE 802.1x/WPA Authenticator and Supplicant (WPA-PSK only).
 endef
 
-define Package/wpad-mesh
-$(call Package/wpad/Default,$(1))
+define Package/wpad-mesh-openssl
+$(call Package/wpad/Default)
+  TITLE+= (with 802.11s mesh and SAE support)
+  DEPENDS+=@PACKAGE_kmod-cfg80211 @(!TARGET_uml||BROKEN) +libopenssl
+  VARIANT:=wpad-mesh-openssl
+  CONFLICTS:=$(filter-out wpad-mesh-openssl ,$(ANY_PROVIDERS))
+  PROVIDES+=wpa-supplicant-mesh wpad-mesh
+endef
+
+define Package/wpad-mesh-wolfssl
+$(call Package/wpad/Default)
   TITLE+= (with 802.11s mesh and SAE support)
-  DEPENDS+=@PACKAGE_kmod-cfg80211 @(!TARGET_uml||BROKEN)
+  DEPENDS+=@PACKAGE_kmod-cfg80211 @(!TARGET_uml||BROKEN) +libwolfssl 
+  VARIANT:=wpad-mesh-wolfssl
+  CONFLICTS:=$(filter-out wpad-mesh-openssl ,$(filter-out wpad-mesh-wolfssl ,$(ANY_PROVIDERS)))
   PROVIDES+=wpa-supplicant-mesh wpad-mesh
 endef
 
@@ -249,49 +298,47 @@ define Package/wpad-mesh/description
  This package contains a minimal IEEE 802.1x/WPA Authenticator and Supplicant (with 802.11s mesh and SAE support).
 endef
 
-define Package/wpad-mesh-openssl
-$(call Package/wpad-mesh,$(1))
-  DEPENDS+=+libopenssl
-  VARIANT:=wpad-mesh-openssl
-endef
-
 Package/wpad-mesh-openssl/description = $(Package/wpad-mesh/description)
-
-define Package/wpad-mesh-wolfssl
-$(call Package/wpad-mesh,$(1))
-  DEPENDS+=+libwolfssl
-  VARIANT:=wpad-mesh-wolfssl
-endef
-
 Package/wpad-mesh-wolfssl/description = $(Package/wpad-mesh/description)
 
-
 define Package/wpa-supplicant/Default
   SECTION:=net
   CATEGORY:=Network
   TITLE:=WPA Supplicant
   URL:=http://hostap.epitest.fi/wpa_supplicant/
   DEPENDS:=$(DRV_DEPENDS)
-  PROVIDES:=wpa-supplicant
-  CONFLICTS:=$(SUPPLICANT_PROVIDERS)
-  SUPPLICANT_PROVIDERS+=$(1)
 endef
 
 define Package/wpa-supplicant
-$(call Package/wpa-supplicant/Default,$(1))
+  $(Package/wpa-supplicant/Default)
   VARIANT:=supplicant-full-internal
+  CONFLICTS:=wpa-supplicant-mini
 endef
 
 define Package/wpa-supplicant-openssl
-$(call Package/wpa-supplicant/Default,$(1))
+  $(Package/wpa-supplicant/Default)
+  CONFLICTS:=$(filter-out wpa-supplicant-wolfssl ,\
+	     $(filter-out wpa-supplicant-openssl ,\
+	     $(filter-out wpa-supplicant-mesh-openssl ,\
+	     $(filter-out wpa-supplicant-mesh-wolfssl ,\
+	     $(SUPPLICANT_ONLY_PROVIDERS)\
+	     ))))
   VARIANT:=supplicant-full-openssl
   DEPENDS+=+libopenssl
+  PROVIDES:=wpa-supplicant
 endef
 
 define Package/wpa-supplicant-wolfssl
-$(call Package/wpa-supplicant/Default,$(1))
+  $(Package/wpa-supplicant/Default)
+  CONFLICTS:=$(filter-out wpa-supplicant-wolfssl ,\
+	     $(filter-out wpa-supplicant-openssl ,\
+	     $(filter-out wpa-supplicant-mesh-openssl ,\
+	     $(filter-out wpa-supplicant-mesh-wolfssl ,\
+	     $(SUPPLICANT_ONLY_PROVIDERS)\
+	     ))))
   VARIANT:=supplicant-full-wolfssl
   DEPENDS+=+libwolfssl
+  PROVIDES:=wpa-supplicant
 endef
 
 define Package/wpa-supplicant/config
@@ -299,61 +346,60 @@ define Package/wpa-supplicant/config
 endef
 
 define Package/wpa-supplicant-p2p
-$(call Package/wpa-supplicant/Default,$(1))
-  TITLE+= (with Wi-Fi P2P support)
-  DEPENDS+=@PACKAGE_kmod-cfg80211
+  $(Package/wpa-supplicant)
+  TITLE:=WPA Supplicant (with Wi-Fi P2P support)
+  DEPENDS:=$(DRV_DEPENDS) @PACKAGE_kmod-cfg80211
+  CONFLICTS:=$(filter-out wpa-supplicant-openssl ,\
+	     $(filter-out wpa-supplicant-wolfssl ,\
+	     $(filter-out wpa-supplicant-p2p ,\
+	     $(filter-out wpa-supplicant-mesh-openssl ,\
+	     $(filter-out wpa-supplicant-mesh-wolfssl ,\
+	     $(SUPPLICANT_ONLY_PROVIDERS)\
+	     )))))
+  VARIANT:=supplicant-p2p-internal
+  PROVIDES:=wpa-supplicant
 endef
 
 define Package/wpa-supplicant-mesh/Default
-$(call Package/wpa-supplicant/Default,$(1))
-  TITLE+= (with 802.11s and SAE)
-  DEPENDS+=@PACKAGE_kmod-cfg80211 @(!TARGET_uml||BROKEN)
-  PROVIDES+=wpa-supplicant-mesh
+  $(Package/wpa-supplicant/Default)
+  TITLE:=WPA Supplicant (with 802.11s and SAE)
+  DEPENDS:=$(DRV_DEPENDS) @PACKAGE_kmod-cfg80211 @(!TARGET_uml||BROKEN)
+  PROVIDES:=wpa-supplicant wpa-supplicant-mesh
 endef
 
 define Package/wpa-supplicant-mesh-openssl
-$(call Package/wpa-supplicant-mesh/Default,$(1))
+  $(Package/wpa-supplicant-mesh/Default)
   VARIANT:=supplicant-mesh-openssl
+  CONFLICTS:=$(filter-out wpa-supplicant-mesh-openssl ,$(SUPPLICANT_ONLY_PROVIDERS))
   DEPENDS+=+libopenssl
 endef
 
 define Package/wpa-supplicant-mesh-wolfssl
-$(call Package/wpa-supplicant-mesh/Default,$(1))
+  $(Package/wpa-supplicant-mesh/Default)
   VARIANT:=supplicant-mesh-wolfssl
+  CONFLICTS:=$(filter-out wpa-supplicant-mesh-openssl ,$(filter-out wpa-supplicant-mesh-wolfssl ,$(SUPPLICANT_ONLY_PROVIDERS)))
   DEPENDS+=+libwolfssl
 endef
 
 define Package/wpa-supplicant-mini
-$(call Package/wpa-supplicant/Default,$(1))
-  TITLE+= (minimal version)
+  $(Package/wpa-supplicant/Default)
+  TITLE:=WPA Supplicant (minimal version)
+  DEPENDS:=$(DRV_DEPENDS)
   VARIANT:=supplicant-mini
+  PROVIDES:=wpa-supplicant
 endef
 
-
-define Package/hostapd-common
-  TITLE:=hostapd/wpa_supplicant common support files
-  SECTION:=net
-  CATEGORY:=Network
-endef
-
-define Package/hostapd-utils
+define Package/wpa-cli
   SECTION:=net
   CATEGORY:=Network
-  TITLE:=IEEE 802.1x Authenticator (utils)
-  URL:=http://hostap.epitest.fi/
-  DEPENDS:=@$(subst $(space),||,$(foreach pkg,$(HOSTAPD_PROVIDERS),PACKAGE_$(pkg)))
-endef
-
-define Package/hostapd-utils/description
- This package contains a command line utility to control the
- IEEE 802.1x/WPA/EAP/RADIUS Authenticator.
+  DEPENDS:=@$(subst $(space),||,$(foreach pkg,$(ANY_SUPPLICANT_PROVIDERS),PACKAGE_$(pkg)))
+  TITLE:=WPA Supplicant command line control utility
 endef
 
-define Package/wpa-cli
+define Package/hostapd-common
+  TITLE:=hostapd/wpa_supplicant common support files
   SECTION:=net
   CATEGORY:=Network
-  DEPENDS:=@$(subst $(space),||,$(foreach pkg,$(SUPPLICANT_PROVIDERS),PACKAGE_$(pkg)))
-  TITLE:=WPA Supplicant command line control utility
 endef
 
 define Package/eapol-test
-- 
2.18.0

