From 8914cfe9767c7517156fe54a74f89416344acffc Mon Sep 17 00:00:00 2001
From: Build system automate <auto-build@example.com>
Date: Thu, 1 Nov 2018 15:51:51 +0100
Subject: [PATCH] package-metadata.pl: also add dependency on missing packages

---
 scripts/package-metadata.pl | 1 -
 1 file changed, 1 deletion(-)

diff --git a/scripts/package-metadata.pl b/scripts/package-metadata.pl
index 53bb45a..73b5015 100755
--- a/scripts/package-metadata.pl
+++ b/scripts/package-metadata.pl
@@ -474,7 +474,6 @@ sub gen_package_mk() {
 				unless (defined($src_dep) && (!$deptype || grep { $_ eq $deptype } @{$src_dep->{buildtypes}})) {
 					warn sprintf "WARNING: Makefile '%s' has a build dependency on '%s', which does not exist\n",
 						$src->{makefile}, $dep.$depsuffix;
-					next;
 				}
 
 				my $depstr = sprintf '$(curdir)/%s/compile', $src_dep->{path}.$depsuffix;
-- 
2.19.1

