From 95be4fd0438cdb098bf04787d7daa8047c1a93cf Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <Michal@Hrusecky.net>
Date: Wed, 23 Jan 2019 22:31:51 +0100
Subject: [PATCH] Disable ABI stuff for now

Signed-off-by: Michal Hrusecky <Michal@Hrusecky.net>

---
 include/package-pack.mk | 14 +++-----------
 1 file changed, 3 insertions(+), 11 deletions(-)

diff --git a/include/package-pack.mk b/include/package-pack.mk
index 0c01112b27..13b5eca9af 100644
--- a/include/package-pack.mk
+++ b/include/package-pack.mk
@@ -193,10 +193,9 @@ _endef=endef
 
 ifeq ($(DUMP),)
   define BuildTarget/ipkg
-    ABIV_$(1):=$(call FormatABISuffix,$(1),$(ABI_VERSION))
     PDIR_$(1):=$(call FeedPackageDir,$(1))
 ifeq ($(CONFIG_USE_APK),)
-    PACK_$(1):=$$(PDIR_$(1))/$(1)$$(ABIV_$(1))_$(VERSION)_$(PKGARCH).ipk
+    PACK_$(1):=$$(PDIR_$(1))/$(1)_$(VERSION)_$(PKGARCH).ipk
 else
     PACK_$(1):=$$(PDIR_$(1))/$(1)$$(ABIV_$(1))-$(VERSION).apk
 endif
@@ -273,17 +272,10 @@ endif
 
     $(STAGING_DIR_ROOT)/stamp/.$(1)_installed: $(PKG_BUILD_DIR)/.pkgdir/$(1).installed
 	mkdir -p $(STAGING_DIR_ROOT)/stamp
-	$(if $(ABI_VERSION),echo '$(ABI_VERSION)' | cmp -s - $(PKG_INFO_DIR)/$(1).version || { \
-		mkdir -p $(PKG_INFO_DIR); \
-		echo '$(ABI_VERSION)' > $(PKG_INFO_DIR)/$(1).version; \
-		$(foreach pkg,$(call SanitizeProvides,$(1),$(PROVIDES)), \
-			cp $(PKG_INFO_DIR)/$(1).version $(PKG_INFO_DIR)/$(pkg).version; \
-		) \
-	} )
 	$(call locked,$(CP) $(PKG_BUILD_DIR)/.pkgdir/$(1)/. $(STAGING_DIR_ROOT)/,root-copy)
 	touch $$@
 
-    Package/$(1)/DEPENDS := $$(foreach dep,$$(filter-out @%,$$(IDEPEND_$(1))),$$(dep)$$(call GetABISuffix,$$(dep)))
+    Package/$(1)/DEPENDS := $$(foreach dep,$$(filter-out @%,$$(IDEPEND_$(1))),$$(dep))
     ifneq ($$(EXTRA_DEPENDS),)
       ifeq ($(CONFIG_USE_APK),)
         Package/$(1)/DEPENDS := $$(call mergelist,$$(Package/$(1)/DEPENDS))
@@ -332,7 +324,7 @@ endif
     endif
 
 $(_define) Package/$(1)/CONTROL
-Package: $(1)$$(ABIV_$(1))
+Package: $(1)
 Version: $(VERSION)
 $$(call addfield,Depends,$$(Package/$(1)/DEPENDS)
 )$$(call addfield,Conflicts,$$(call mergelist,$(CONFLICTS))
-- 
2.52.0

