From be622a150f1275b7d0d69826a8649a1eec21db17 Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <Michal@Hrusecky.net>
Date: Wed, 23 Jan 2019 22:31:51 +0100
Subject: [PATCH] Disable ABI stuff for now

Signed-off-by: Michal Hrusecky <Michal@Hrusecky.net>
---
 include/package-ipkg.mk | 1 -
 1 file changed, 1 deletion(-)

diff --git a/include/package-ipkg.mk b/include/package-ipkg.mk
index a906f8f96f..b063eb4b29 100644
--- a/include/package-ipkg.mk
+++ b/include/package-ipkg.mk
@@ -93,7 +93,6 @@ _endef=endef
 
 ifeq ($(DUMP),)
   define BuildTarget/ipkg
-    ABIV_$(1):=$(call GetABISuffix,$(1))
     PDIR_$(1):=$(call FeedPackageDir,$(1))
     IPKG_$(1):=$$(PDIR_$(1))/$(1)$$(ABIV_$(1))_$(VERSION)_$(PKGARCH).ipk
     IDIR_$(1):=$(PKG_BUILD_DIR)/ipkg-$(PKGARCH)/$(1)
-- 
2.20.1

