From 075f3c24e415e46be888a115fe7b307d415e2a56 Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <Michal@Hrusecky.net>
Date: Wed, 23 Jan 2019 22:31:51 +0100
Subject: [PATCH] Disable ABI stuff for now

Signed-off-by: Michal Hrusecky <Michal@Hrusecky.net>
---
 include/package-ipkg.mk | 1 -
 1 file changed, 1 deletion(-)

diff --git a/include/package-ipkg.mk b/include/package-ipkg.mk
index c374faa647..6ab798c613 100644
--- a/include/package-ipkg.mk
+++ b/include/package-ipkg.mk
@@ -144,7 +144,6 @@ ifeq ($(DUMP),)
 
     $(STAGING_DIR_ROOT)/stamp/.$(1)_installed: $(PKG_BUILD_DIR)/.pkgdir/$(1).installed
 	mkdir -p $(STAGING_DIR_ROOT)/stamp
-	$(if $(ABI_VERSION),echo '$(ABI_VERSION)' | cmp -s - $(PKG_INFO_DIR)/$(1).version || echo '$(ABI_VERSION)' > $(PKG_INFO_DIR)/$(1).version)
 	$(call locked,$(CP) $(PKG_BUILD_DIR)/.pkgdir/$(1)/. $(STAGING_DIR_ROOT)/,root-copy)
 	touch $$@
 
-- 
2.20.1

