From f9a51635b16ce712e50fb0504df2983d1b32058f Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <Michal@Hrusecky.net>
Date: Thu, 28 Feb 2019 17:47:07 +0100
Subject: [PATCH] opkg: Add wget dependency

---
 package/system/opkg/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package/system/opkg/Makefile b/package/system/opkg/Makefile
index 68d379653b..eb86915a80 100644
--- a/package/system/opkg/Makefile
+++ b/package/system/opkg/Makefile
@@ -42,7 +42,7 @@ define Package/opkg
   SECTION:=base
   CATEGORY:=Base system
   TITLE:=opkg package manager
-  DEPENDS:=+uclient-fetch +libpthread +libubox +switch-branch
+  DEPENDS:=+uclient-fetch +wget +libpthread +libubox +switch-branch
   URL:=$(PKG_SOURCE_URL)
   MENU:=1
 endef
-- 
2.20.1

