From 44a7cc567bef06b2f1116a7c11e36c535339d078 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Mon, 26 Jul 2021 15:34:29 +0200
Subject: [PATCH 2/2] mpc85xx: force kernel 5.10

---
 target/linux/mpc85xx/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/target/linux/mpc85xx/Makefile b/target/linux/mpc85xx/Makefile
index 5e3ffb8b3c..0ddd71a50e 100644
--- a/target/linux/mpc85xx/Makefile
+++ b/target/linux/mpc85xx/Makefile
@@ -11,7 +11,7 @@ CPU_TYPE:=8540
 FEATURES:=squashfs ramdisk
 SUBTARGETS:=p1010 p1020 p2020
 
-KERNEL_PATCHVER:=5.4
+KERNEL_PATCHVER:=5.10
 KERNEL_TESTING_PATCHVER:=5.10
 
 KERNELNAME:=zImage
-- 
2.30.2

