From a2532b60aac6f728810b9584224da89ad22b13cc Mon Sep 17 00:00:00 2001
From: Richard Muzik <richard.muzik@nic.cz>
Date: Wed, 26 Jun 2024 13:37:27 +0200
Subject: [PATCH] elfutils: fix build with GCC8

---
 package/libs/elfutils/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package/libs/elfutils/Makefile b/package/libs/elfutils/Makefile
index f7364c36be..1464db7fb9 100644
--- a/package/libs/elfutils/Makefile
+++ b/package/libs/elfutils/Makefile
@@ -87,7 +87,7 @@ TARGET_CFLAGS += \
 	-Wno-unused-result \
 	-Wno-format-nonliteral
 
-ifneq ($(CONFIG_GCC_USE_VERSION_11),y)
+ifneq ($(filter y,$(CONFIG_GCC_USE_VERSION_8)$(CONFIG_GCC_USE_VERSION_11)),y)
 TARGET_CFLAGS += \
 	-Wno-error=use-after-free
 endif
-- 
2.45.2

