From c0d946f931236e39c6e9bea666f2212d5013ceea Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Tue, 7 Jun 2022 11:29:59 +0200
Subject: [PATCH] HACK: Enable Marvell switches to every router we have

---
 target/linux/generic/config-5.15 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/target/linux/generic/config-5.15 b/target/linux/generic/config-5.15
index 8146161d08..d398402e5b 100644
--- a/target/linux/generic/config-5.15
+++ b/target/linux/generic/config-5.15
@@ -4021,7 +4021,7 @@ CONFIG_NET_CORE=y
 # CONFIG_NET_DSA_MV88E6131 is not set
 # CONFIG_NET_DSA_MV88E6171 is not set
 # CONFIG_NET_DSA_MV88E6352 is not set
-# CONFIG_NET_DSA_MV88E6XXX is not set
+CONFIG_NET_DSA_MV88E6XXX=y
 # CONFIG_NET_DSA_MV88E6XXX_NEED_PPU is not set
 # CONFIG_NET_DSA_MV88E6XXX_PTP is not set
 # CONFIG_NET_DSA_QCA8K is not set
-- 
2.34.1

