From b9e5c80571e702ffdd137fac6315f7fb691d860c Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Sun, 16 Oct 2022 00:02:54 +0200
Subject: [PATCH] node: drop depends from Configuration menu

---
 node/Makefile | 1 -
 1 file changed, 1 deletion(-)

diff --git a/node/Makefile b/node/Makefile
index 5333ff1f..0eb7ea03 100644
--- a/node/Makefile
+++ b/node/Makefile
@@ -83,7 +83,6 @@ endef
 
 define Package/node/config
 	menu "Configuration"
-		depends on PACKAGE_node
 
 	choice
 		prompt "Version Selection"
-- 
2.34.1

