Index: backports-6.6.15/net/wireless/nl80211.c
===================================================================
--- backports-6.6.15.orig/net/wireless/nl80211.c
+++ backports-6.6.15/net/wireless/nl80211.c
@@ -19072,7 +19072,7 @@ void nl80211_common_reg_change_event(enu
 
 	rcu_read_lock();
 	genlmsg_multicast_allns(&nl80211_fam, msg, 0,
-				NL80211_MCGRP_REGULATORY, GFP_ATOMIC);
+				NL80211_MCGRP_REGULATORY);
 	rcu_read_unlock();
 
 	return;
@@ -19816,7 +19816,7 @@ void nl80211_send_beacon_hint_event(stru
 
 	rcu_read_lock();
 	genlmsg_multicast_allns(&nl80211_fam, msg, 0,
-				NL80211_MCGRP_REGULATORY, GFP_ATOMIC);
+				NL80211_MCGRP_REGULATORY);
 	rcu_read_unlock();
 
 	return;
