From 378fc61c7bf2378333b95e18f01a0249c942cb37 Mon Sep 17 00:00:00 2001
From: Tomas Zak <tomas.zak@turris.com>
Date: Fri, 29 Sep 2023 21:00:06 +0200
Subject: [PATCH] packages: python/numpy: Bump to version 1.24.3

Bump numpy to version 1.24.3.
---
 lang/python/numpy/Makefile                    | 58 +++++++++++++++++--

diff --git a/lang/python/numpy/Makefile b/lang/python/numpy/Makefile
index 6b417a775..e4eac68ec 100644
--- a/lang/python/numpy/Makefile
+++ b/lang/python/numpy/Makefile
@@ -6,11 +6,11 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=numpy
-PKG_VERSION:=1.23.3
+PKG_VERSION:=1.24.3
 PKG_RELEASE:=$(AUTORELEASE)
 
 PYPI_NAME:=$(PKG_NAME)
-PKG_HASH:=51bf49c0cd1d52be0a240aa66f3458afc4b95d8993d2d04f0d91fa60c10af6cd
+PKG_HASH:=ab344f1bf21f140adab8e47fdbc7c35a477dc01408791f8ba00d018dd0bc5155
 
 PKG_MAINTAINER:=Alexandru Ardelean <ardeleanalex@gmail.com>
 
-- 
2.42.0

