From de3663674e5a8aa2a0f1246cdffe1a6ffd123d15 Mon Sep 17 00:00:00 2001
From: Michal Vasilek <michal.vasilek@nic.cz>
Date: Tue, 21 Jun 2022 17:46:36 +0200
Subject: [PATCH 1/5] libgd: install pkgconfig file

---
 libs/libgd/Makefile | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/libs/libgd/Makefile b/libs/libgd/Makefile
index 1c32fa10c..0b2c998f6 100644
--- a/libs/libgd/Makefile
+++ b/libs/libgd/Makefile
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=libgd
 PKG_VERSION:=2.2.5
-PKG_RELEASE:=2
+PKG_RELEASE:=3
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
 PKG_SOURCE_URL:=https://github.com/$(PKG_NAME)/$(PKG_NAME)/releases/download/gd-$(PKG_VERSION)/
@@ -104,6 +104,8 @@ define Build/InstallDev
 	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libgd.{a,la,so*} $(1)/usr/lib/
 	$(INSTALL_DIR) $(2)/bin
 	$(LN) ../../usr/bin/gdlib-config $(2)/bin/
+	$(INSTALL_DIR) $(1)/usr/lib/pkgconfig
+	$(CP) $(PKG_INSTALL_DIR)/usr/lib/pkgconfig/gdlib.pc $(1)/usr/lib/pkgconfig/gdlib.pc
 endef
 
 define Package/libgd/install
-- 
2.36.1

