From 6ceafc313752b421e4b3938b571dc0dccd84d552 Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <michal.hrusecky@nic.cz>
Date: Thu, 24 May 2018 12:46:40 +0200
Subject: [PATCH] luajit: Enable build on arm64

Signed-off-by: Michal Hrusecky <michal.hrusecky@nic.cz>
---
 lang/luajit/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lang/luajit/Makefile b/lang/luajit/Makefile
index e5dfa95..334cbb6 100644
--- a/lang/luajit/Makefile
+++ b/lang/luajit/Makefile
@@ -26,7 +26,7 @@ define Package/luajit
  CATEGORY:=Languages
  TITLE:=LuaJIT
  URL:=http://www.luajit.org
- DEPENDS:=@(i386||x86_64||arm||armeb||powerpc||mips||mipsel)
+ DEPENDS:=@(i386||x86_64||arm||armeb||powerpc||mips||mipsel||aarch64)
 endef
 
 define Package/luajit/description
-- 
2.17.0

