From 90eea2dc3733a147d1fedfe1996deb3c377a94fa Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <michal.hrusecky@nic.cz>
Date: Wed, 2 May 2018 16:10:06 +0200
Subject: [PATCH] openssh: Enable pthread hack when compiling with pam

Signed-off-by: Michal Hrusecky <michal.hrusecky@nic.cz>
---
 net/openssh/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/net/openssh/Makefile b/net/openssh/Makefile
index afdd602..3098481 100644
--- a/net/openssh/Makefile
+++ b/net/openssh/Makefile
@@ -189,6 +189,7 @@ CONFIGURE_VARS += LD="$(TARGET_CC)"
 
 ifeq ($(BUILD_VARIANT),with-pam)
 TARGET_LDFLAGS += -lpthread
+TARGET_CFLAGS += -DUNSUPPORTED_POSIX_THREADS_HACK=1
 endif
 
 define Build/Compile
-- 
2.16.3

