From 36a18cd0c22f3618cd1414eb79609037a19e1207 Mon Sep 17 00:00:00 2001
From: Michal Vasilek <michal.vasilek@nic.cz>
Date: Tue, 21 Jun 2022 17:57:04 +0200
Subject: [PATCH 2/5] cyrus-sasl: install pkgconfig file

---
 libs/cyrus-sasl/Makefile | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/libs/cyrus-sasl/Makefile b/libs/cyrus-sasl/Makefile
index 04796f3b3..afabd402b 100644
--- a/libs/cyrus-sasl/Makefile
+++ b/libs/cyrus-sasl/Makefile
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=cyrus-sasl
 PKG_VERSION:=2.1.27
-PKG_RELEASE:=2
+PKG_RELEASE:=3
 
 PKG_MAINTAINER:=W. Michael Petullo <mike@flyn.org>
 
@@ -101,6 +101,8 @@ define Build/InstallDev
 	ln -sf libsasl2.so $(1)/usr/lib/libsasl.so
 	$(INSTALL_DIR) $(1)/usr/lib/sasl2
 	$(CP) $(PKG_INSTALL_DIR)/usr/lib/sasl2/lib*.{a,so*} $(1)/usr/lib/sasl2/
+	$(INSTALL_DIR) $(1)/usr/lib/pkgconfig/
+	$(CP) $(PKG_INSTALL_DIR)/usr/lib/pkgconfig/libsasl2.pc $(1)/usr/lib/pkgconfig/libsasl2.pc
 endef
 
 define Package/libsasl2/install
-- 
2.36.1

