From a02d444ceba0a5972dd76b2412199edbf35cb072 Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <michal.hrusecky@nic.cz>
Date: Wed, 2 May 2018 16:10:06 +0200
Subject: [PATCH 06/12] openssh: Enable pthread hack when compiling with pam

Signed-off-by: Michal Hrusecky <michal.hrusecky@nic.cz>
---
 net/openssh/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/net/openssh/Makefile b/net/openssh/Makefile
index 639ce5f..02882df 100644
--- a/net/openssh/Makefile
+++ b/net/openssh/Makefile
@@ -194,6 +194,7 @@ CONFIGURE_VARS += LD="$(TARGET_CC)"
 
 ifeq ($(BUILD_VARIANT),with-pam)
 TARGET_LDFLAGS += -lpthread
+TARGET_CFLAGS += -DUNSUPPORTED_POSIX_THREADS_HACK=1
 endif
 
 define Build/Compile
-- 
2.19.1

