From 73e4f512717613ecf1b690afd7e339acecf16ea4 Mon Sep 17 00:00:00 2001
From: Stepan Henek <stepan.henek@nic.cz>
Date: Wed, 22 Aug 2018 13:05:02 +0200
Subject: [PATCH] python-jinja2: Add host build

Signed-off-by: Stepan Henek <stepan.henek@nic.cz>
---
 lang/python/Jinja2/Makefile | 42 ++++++++++++++++++++++++++++++++-----
 1 file changed, 37 insertions(+), 5 deletions(-)

diff --git a/lang/python/Jinja2/Makefile b/lang/python/Jinja2/Makefile
index c1d1de6..8f060ef 100644
--- a/lang/python/Jinja2/Makefile
+++ b/lang/python/Jinja2/Makefile
@@ -17,17 +17,29 @@ PKG_LICENSE_FILES:=LICENSE
 PKG_MAINTAINER:=Daniel Golle <daniel@makrotopia.org>
 
 PKG_BUILD_DIR:=$(BUILD_DIR)/$(BUILD_VARIANT)-$(PKG_NAME)-$(PKG_VERSION)
+HOST_BUILD_DIR:=$(BUILD_DIR_HOST)/$(BUILD_VARIANT)-libsass-$(PKG_VERSION)
 PKG_UNPACK=$(HOST_TAR) -C $(PKG_BUILD_DIR) --strip-components=1 -xzf $(DL_DIR)/$(PKG_SOURCE)
+HOST_UNPACK:=$(HOST_TAR) -C $(HOST_BUILD_DIR) --strip-components=1 -xzf $(DL_DIR)/$(PKG_SOURCE)
+
+PKG_BUILD_DEPENDS:=python3
+HOST_BUILD_DEPENDS:=python3/host MarkupSafe/host
 
 include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/host-build.mk
 include ../python3-package.mk
+include ../python3-host.mk
 include ../python-package.mk
+include ../python-host.mk
 
-define Package/python3-jinja2
+define Package/python-jinja2/Default
   SECTION:=lang
   CATEGORY:=Languages
   SUBMENU:=Python
   URL:=http://jinja.pocoo.org/
+endef
+
+define Package/python3-jinja2
+  $(call Package/python-jinja2/Default)
   TITLE:=python3-jinja2
   DEPENDS:=+python3 +python3-markupsafe
   VARIANT:=python3
@@ -40,10 +52,7 @@ environment, widely used and BSD licensed.
 endef
 
 define Package/python-jinja2
-  SECTION:=lang
-  CATEGORY:=Languages
-  SUBMENU:=Python
-  URL:=http://jinja.pocoo.org/
+  $(call Package/python-jinja2/Default)
   TITLE:=python-jinja2
   DEPENDS:=+python +python-markupsafe
   VARIANT:=python
@@ -55,14 +64,37 @@ unicode support, an optional integrated sandboxed execution
 environment, widely used and BSD licensed.
 endef
 
+define Package/python3-jinja2/description
+$(call Package/python-jinja2/description)
+.
+(Variant for Python3)
+endef
+
 ifeq ($(BUILD_VARIANT),python)
 define remove_async
 	rm -rf $(PKG_BUILD_DIR)/jinja2/async*
 endef
 Hooks/Prepare/Post += remove_async
+
+define remove_host_async
+	rm -rf $(HOST_BUILD_DIR)/jinja2/async*
+endef
+Hooks/HostPrepare/Post += remove_host_async
 endif
 
+define Host/Compile
+	$(call Build/Compile/HostPyMod,,install --prefix="" --root="$(STAGING_DIR_HOSTPKG)")
+	$(call Build/Compile/HostPy3Mod,,install --prefix="" --root="$(STAGING_DIR_HOSTPKG)")
+endef
+
+Host/Install:=
+
+$(eval $(call HostBuild))
+
 $(eval $(call Py3Package,python3-jinja2))
 $(eval $(call BuildPackage,python3-jinja2))
+$(eval $(call BuildPackage,python3-jinja2-src))
+
 $(eval $(call PyPackage,python-jinja2))
 $(eval $(call BuildPackage,python-jinja2))
+$(eval $(call BuildPackage,python-jinja2-src))
-- 
2.19.1

