From 4b837263f8d17433100e235a5c2afc251dc4038c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Karel=20Ko=C4=8D=C3=AD?= <karel.koci@nic.cz>
Date: Mon, 11 Mar 2019 07:04:23 +0100
Subject: [PATCH] syslog-ng: add missing libmaxminddb dependency

---
 admin/syslog-ng/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/admin/syslog-ng/Makefile b/admin/syslog-ng/Makefile
index 57d3123..7182882 100644
--- a/admin/syslog-ng/Makefile
+++ b/admin/syslog-ng/Makefile
@@ -27,7 +27,7 @@ define Package/syslog-ng
   CATEGORY:=Administration
   TITLE:=A powerful syslog daemon
   URL:=https://www.syslog-ng.com/products/open-source-log-management/
-  DEPENDS:=+libpcre +glib2 +libopenssl +libpthread +librt +zlib +libdbi +libjson-c +libcurl +libuuid
+  DEPENDS:=+libpcre +glib2 +libopenssl +libpthread +librt +zlib +libdbi +libjson-c +libcurl +libuuid +libmaxminddb
 endef
 
 define Package/syslog-ng/description
-- 
2.23.0

