From 54f2f0d15b27dee458bc8e6c5841345b3c15e871 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Karel=20Ko=C4=8D=C3=AD?= <karel.koci@nic.cz>
Date: Mon, 18 Mar 2019 15:20:30 +0100
Subject: [PATCH] python-influxdb: update to 5.2.2

---
 lang/python/python-influxdb/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/lang/python/python-influxdb/Makefile b/lang/python/python-influxdb/Makefile
index aa9d5c747..221db928f 100644
--- a/lang/python/python-influxdb/Makefile
+++ b/lang/python/python-influxdb/Makefile
@@ -5,13 +5,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python-influxdb
-PKG_VERSION:=5.2.1
+PKG_VERSION:=5.2.2
 PKG_RELEASE:=1
 PKG_MAINTAINER:=Karel Kočí <cynerd@email.cz>
 
 PKG_SOURCE_URL:=https://pypi.python.org/packages/source/i/influxdb/
 PKG_SOURCE:=influxdb-$(PKG_VERSION).tar.gz
-PKG_HASH:=75d96de25d0d4e9e66e155f64dc9dc2a48de74ac4e77e3a46ad881fba772e3b6
+PKG_HASH:=afeff28953a91b4ea1aebf9b5b8258a4488d0e49e2471db15ea43fd2c8533143
 PKG_LICENSE:=MIT
 PKG_BUILD_DIR:=$(BUILD_DIR)/influxdb-$(PKG_VERSION)
 
-- 
2.21.0

