From d768be35346f5c99c73c1f9f949607deffa5e74d Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Thu, 12 Sep 2019 00:20:11 +0200
Subject: [PATCH] btrfs-progs: Update to version 5.2.2

Signed-off-by: Josef Schlehofer <pepe.schlehofer@gmail.com>
---
 utils/btrfs-progs/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/utils/btrfs-progs/Makefile b/utils/btrfs-progs/Makefile
index 3ad77c981..2ea0aadef 100644
--- a/utils/btrfs-progs/Makefile
+++ b/utils/btrfs-progs/Makefile
@@ -6,12 +6,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=btrfs-progs
-PKG_VERSION:=5.2.1
+PKG_VERSION:=5.2.2
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-v$(PKG_VERSION).tar.xz
 PKG_SOURCE_URL:=@KERNEL/linux/kernel/people/kdave/btrfs-progs
-PKG_HASH:=36ac4a0198ffff79d5800c537ea4b19769a8fd3ad870f75413d25b20e2d83233
+PKG_HASH:=821321dbf17087e1172023fa35656ce52d342fbfe210fb8ea01fc57b65dfb1c6
 PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-v$(PKG_VERSION)
 
 PKG_MAINTAINER:=Karel Kočí <karel.koci@nic.cz>
-- 
2.20.1

