From f8a5483bd10c5ccb11c5fb238569e675d254c1d3 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Thu, 16 Jan 2020 12:45:15 +0100
Subject: [PATCH] libseccomp: add seccomp-syscalls.h to InstallDev

Signed-off-by: Josef Schlehofer <pepe.schlehofer@gmail.com>
---
 libs/libseccomp/Makefile | 13 ++++---------
 1 file changed, 4 insertions(+), 9 deletions(-)

diff --git a/libs/libseccomp/Makefile b/libs/libseccomp/Makefile
index ecc075a8c..fd479a8e6 100644
--- a/libs/libseccomp/Makefile
+++ b/libs/libseccomp/Makefile
@@ -59,15 +59,10 @@ endef
 
 define Build/InstallDev
 	$(INSTALL_DIR) $(1)/usr/include $(1)/usr/lib/pkgconfig
-	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/include/seccomp.h \
-		$(1)/usr/include/
-	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/libseccomp.{a,so*} \
-		$(1)/usr/lib/
-	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/pkgconfig/libseccomp.pc \
-		$(1)/usr/lib/pkgconfig/
+	$(CP) $(PKG_INSTALL_DIR)/usr/include/seccomp.h $(1)/usr/include/
+	$(CP) $(PKG_INSTALL_DIR)/usr/include/seccomp-syscalls.h $(1)/usr/include/
+	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libseccomp.{a,so*} $(1)/usr/lib/
+	$(CP) $(PKG_INSTALL_DIR)/usr/lib/pkgconfig/libseccomp.pc $(1)/usr/lib/pkgconfig/
 endef
 
 define Package/libseccomp/install
-- 
2.24.1

