From a7f3cb7956507a790c768f8f89cd2f2d47097b6b Mon Sep 17 00:00:00 2001
From: Richard Muzik <richard.muzik@nic.cz>
Date: Fri, 26 Jul 2024 11:39:41 +0200
Subject: [PATCH] tvheadend: fix init

---
 multimedia/tvheadend/files/tvheadend.init | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/multimedia/tvheadend/files/tvheadend.init b/multimedia/tvheadend/files/tvheadend.init
index 44e71c44a..28709016f 100644
--- a/multimedia/tvheadend/files/tvheadend.init
+++ b/multimedia/tvheadend/files/tvheadend.init
@@ -63,7 +63,7 @@ load_uci_config() {
 	config_get_bool ipv6 server ipv6 0
 	[ "$ipv6" -eq 1 ] && procd_append_param command --ipv6
 	config_get bindaddr server bindaddr
-	[ -z "$bindaddr" ] && bindaddr=$(uci get network.lan.ipaddr)
+	[ -z "$bindaddr" ] && bindaddr=$(uci get network.lan.ipaddr | sed 's/\/.*//')
 	[ -n "$bindaddr" ] && procd_append_param command --bindaddr "$bindaddr"
 	config_get http_port server http_port
 	[ -n "$http_port" ] && procd_append_param command --http_port "$http_port"
-- 
2.45.2

