From e3c3c3f28857937794f7355e23b4f812325b89b3 Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <michal.hrusecky@turris.com>
Date: Fri, 6 Dec 2024 09:22:48 +0100
Subject: [PATCH] miniupnp: Add dependencies on firewall

---
 net/miniupnpd/Makefile | 2 ++
 1 file changed, 2 insertions(+), 0 deletion(-)

diff --git a/net/miniupnpd/Makefile b/net/miniupnpd/Makefile
index 1526e922c..0cdefcad6 100644
--- a/net/miniupnpd/Makefile
+++ b/net/miniupnpd/Makefile
@@ -42,6 +42,7 @@ endef
 define Package/miniupnpd-iptables
   $(call Package/miniupnpd/Default)
   DEPENDS+= \
+	+firewall \
 	+IPV6:ip6tables \
 	+IPV6:libip6tc \
 	+iptables \
@@ -54,6 +55,7 @@ endef
 define Package/miniupnpd-nftables
   $(call Package/miniupnpd/Default)
   DEPENDS+= \
+	+firewall4 \
 	+libnftnl
   TITLE+= (nftables)
   VARIANT:=nftables
-- 
2.47.1

