From 3854923f78610978f90b6ad931823443db14302f Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <michal.hrusecky@turris.com>
Date: Fri, 6 Dec 2024 09:22:48 +0100
Subject: [PATCH] miniupnp: Add dependencies on firewall

---
 net/miniupnpd/Makefile | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/net/miniupnpd/Makefile b/net/miniupnpd/Makefile
index 6373f309c..a6315ab0e 100644
--- a/net/miniupnpd/Makefile
+++ b/net/miniupnpd/Makefile
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=miniupnpd
 PKG_VERSION:=2.3.3
-PKG_RELEASE:=1
+PKG_RELEASE:=2
 
 PKG_SOURCE_URL:=https://miniupnp.tuxfamily.org/files
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
@@ -42,6 +42,7 @@ endef
 define Package/miniupnpd-iptables
   $(call Package/miniupnpd/Default)
   DEPENDS+= \
+	+firewall \
 	+IPV6:ip6tables \
 	+IPV6:libip6tc \
 	+iptables \
@@ -54,6 +55,7 @@ endef
 define Package/miniupnpd-nftables
   $(call Package/miniupnpd/Default)
   DEPENDS+= \
+	+firewall4 \
 	+libnftnl
   TITLE+= (nftables)
   VARIANT:=nftables
-- 
2.47.1

