From 1d10c1fd5afbc1db00774bf026f0e25a92c6112c Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Mon, 9 Sep 2019 08:38:12 +0200
Subject: [PATCH 2/2] dovecot-pigeonhole: Update to version 0.4.24.2

Fix CVE-2019-11500

Signed-off-by: Josef Schlehofer <pepe.schlehofer@gmail.com>
---
 mail/pigeonhole/Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/mail/pigeonhole/Makefile b/mail/pigeonhole/Makefile
index 9add29e78..332b0a38d 100644
--- a/mail/pigeonhole/Makefile
+++ b/mail/pigeonhole/Makefile
@@ -8,16 +8,16 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=dovecot-pigeonhole
-PKG_VERSION_PLUGIN:=0.4.21
+PKG_VERSION_PLUGIN:=0.4.24.2
 PKG_VERSION_DOVECOT:=$(shell make --no-print-directory -C ../dovecot/ val.PKG_VERSION V=s)
 PKG_VERSION:=$(PKG_VERSION_DOVECOT)-$(PKG_VERSION_PLUGIN)
-PKG_RELEASE:=2
+PKG_RELEASE:=1
 
 DOVECOT_VERSION:=2.2
 
 PKG_SOURCE:=dovecot-$(DOVECOT_VERSION)-pigeonhole-$(PKG_VERSION_PLUGIN).tar.gz
 PKG_SOURCE_URL:=https://pigeonhole.dovecot.org/releases/$(DOVECOT_VERSION)
-PKG_HASH:=4ae09cb788c5334d167f5a89ee70b0616c3231e5904ad258ce408e4953cfdd6a
+PKG_HASH:=1a79e94447a64af33e2248a9ea60d0553d543bd05491a348a6a0632d2ea25bca
 PKG_LICENSE:=LGPL-2.1
 PKG_LICENSE_FILES:=COPYING COPYING.LGPL
 
-- 
2.20.1

