From 43ab6d5fcb1fa753e5c11a6cb7cb9a97b2c83d50 Mon Sep 17 00:00:00 2001
From: Richard Muzik <richard.muzik@nic.cz>
Date: Mon, 28 Apr 2025 12:23:47 +0200
Subject: [PATCH 2/2] python-twisted: update to version 24.7.0

---
 lang/python/python-twisted/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/lang/python/python-twisted/Makefile b/lang/python/python-twisted/Makefile
index 79f2f2a23..32f396f2f 100644
--- a/lang/python/python-twisted/Makefile
+++ b/lang/python/python-twisted/Makefile
@@ -9,12 +9,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python-twisted
-PKG_VERSION:=23.10.0
+PKG_VERSION:=24.7.0
 PKG_RELEASE:=1
 
 PYPI_NAME:=Twisted
 PYPI_SOURCE_NAME:=twisted
-PKG_HASH:=987847a0790a2c597197613686e2784fd54167df3a55d0fb17c8412305d76ce5
+PKG_HASH:=5a60147f044187a127ec7da96d170d49bcce50c6fd36f594e60f4587eff4d394
 
 PKG_BUILD_DEPENDS:=libtirpc
 
-- 
2.49.0

