From 0146a50b4bce2ae80eb13827f411897674eda920 Mon Sep 17 00:00:00 2001
From: Richard Muzik <richard.muzik@nic.cz>
Date: Thu, 6 Mar 2025 13:02:14 +0100
Subject: [PATCH] avrdude: add missing dependency for libintl

---
 utils/avrdude/Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/utils/avrdude/Makefile b/utils/avrdude/Makefile
index b45909b2a..d8f2801a8 100644
--- a/utils/avrdude/Makefile
+++ b/utils/avrdude/Makefile
@@ -22,6 +22,7 @@ PKG_LICENSE_FILES:=COPYING
 
 include $(INCLUDE_DIR)/package.mk
 include $(INCLUDE_DIR)/cmake.mk
+include $(INCLUDE_DIR)/nls.mk
 
 define Package/avrdude
   SECTION:=utils
@@ -29,7 +30,7 @@ define Package/avrdude
   SUBMENU:=Microcontroller programming
   TITLE:=AVR Downloader/UploaDEr
   URL:=https://github.com/avrdudes/avrdude
-  DEPENDS:=+libncurses +libreadline +libusb-1.0 +libftdi1 +libelf +libgpiod
+  DEPENDS:=+libncurses +libreadline +libusb-1.0 +libftdi1 +libelf +libgpiod $(INTL_DEPENDS)
 endef
 
 define Package/avrdude/description
-- 
2.48.1

