From 29ccd9c3cac9f258e08d15f98db208ea5cb5021c Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <michal.hrusecky@turris.com>
Date: Thu, 9 Apr 2026 12:36:50 +0200
Subject: [PATCH] samba4: Update to the latest 4.22 version - 4.22.9

---
 net/samba4/Makefile                           | 4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/net/samba4/Makefile b/net/samba4/Makefile
index eb7d3d807..b2352d070 100644
--- a/net/samba4/Makefile
+++ b/net/samba4/Makefile
@@ -2,7 +2,7 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=samba
-PKG_VERSION:=4.22.7
+PKG_VERSION:=4.22.9
 PKG_RELEASE:=3
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
@@ -13,7 +13,7 @@ PKG_SOURCE_URL:= \
 		http://www.nic.funet.fi/index/samba/pub/samba/stable/ \
 		http://samba.mirror.bit.nl/samba/ftp/stable/ \
 		https://download.samba.org/pub/samba/stable/
-PKG_HASH:=12195811d4542f661536e9055b44d58c53020412beafaae205e227bf72f6a497
+PKG_HASH:=15f8fa92d235573880353347167c37f3988c977adcf2e057c4a8f08105477bfc
 
 PKG_BUILD_FLAGS:=gc-sections
 
--
2.53.0

