From 9d9ae2e63f3474b10987e32557c60b52f535a10c Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <michal.hrusecky@turris.com>
Date: Wed, 8 Apr 2026 11:03:08 +0200
Subject: [PATCH] openssh: Update to version 10.3p1

---
 net/openssh/Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/net/openssh/Makefile b/net/openssh/Makefile
index fe6b8e040..5f7499482 100644
--- a/net/openssh/Makefile
+++ b/net/openssh/Makefile
@@ -8,14 +8,14 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=openssh
-PKG_REALVERSION:=10.2p1
-PKG_VERSION:=10.2_p1
+PKG_REALVERSION:=10.3p1
+PKG_VERSION:=10.3_p1
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_REALVERSION).tar.gz
 PKG_SOURCE_URL:=https://cdn.openbsd.org/pub/OpenBSD/OpenSSH/portable/ \
 		https://ftp.openbsd.org/pub/OpenBSD/OpenSSH/portable/
-PKG_HASH:=ccc42c0419937959263fa1dbd16dafc18c56b984c03562d2937ce56a60f798b2
+PKG_HASH:=56682a36bb92dcf4b4f016fd8ec8e74059b79a8de25c15d670d731e7d18e45f4
 PKG_BUILD_DIR:=$(BUILD_DIR)/$(if $(BUILD_VARIANT),$(PKG_NAME)-$(BUILD_VARIANT)/)$(PKG_NAME)-$(PKG_REALVERSION)
 
 PKG_LICENSE:=BSD ISC
-- 
2.53.0

