From 5720ab4e321bd7f882d2edf63104f0486318781f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marek=20Beh=C3=BAn?= <kabel@kernel.org>
Date: Mon, 30 Jun 2025 10:31:46 +0200
Subject: [PATCH] WIP: drm/mipi_dbi: Allow setting max_speed_hz via debugfs
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Marek Behún <kabel@kernel.org>
---
 drivers/gpu/drm/drm_mipi_dbi.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/gpu/drm/drm_mipi_dbi.c b/drivers/gpu/drm/drm_mipi_dbi.c
index e90f0bf89..335d1c129 100644
--- a/drivers/gpu/drm/drm_mipi_dbi.c
+++ b/drivers/gpu/drm/drm_mipi_dbi.c
@@ -1465,6 +1465,8 @@ void mipi_dbi_debugfs_init(struct drm_minor *minor)
 		mode |= S_IRUGO;
 	debugfs_create_file("command", mode, minor->debugfs_root, dbidev,
 			    &mipi_dbi_debugfs_command_fops);
+
+	debugfs_create_u32("max_speed_hz", 0644, minor->debugfs_root, &dbidev->dbi.spi->max_speed_hz);
 }
 EXPORT_SYMBOL(mipi_dbi_debugfs_init);
 
-- 
2.49.0

