From 5dc3edc39c487788d62a005f78a1d4ea2881314d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marek=20Beh=C3=BAn?= <kabel@kernel.org>
Date: Fri, 22 Aug 2025 12:27:35 +0200
Subject: [PATCH] clk: qcom: gcc-ipq9574: Add various different SPI speeds
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Marek Behún <kabel@kernel.org>
---
 drivers/clk/qcom/gcc-ipq9574.c | 51 ++++++++++++++++++++++++++++++++++
 1 file changed, 51 insertions(+)

diff --git a/drivers/clk/qcom/gcc-ipq9574.c b/drivers/clk/qcom/gcc-ipq9574.c
index 0df6e3407..0b8255fda 100644
--- a/drivers/clk/qcom/gcc-ipq9574.c
+++ b/drivers/clk/qcom/gcc-ipq9574.c
@@ -466,11 +466,62 @@ static struct clk_rcg2 blsp1_qup1_i2c_apps_clk_src = {
 
 static const struct freq_tbl ftbl_blsp1_qup_spi_apps_clk_src[] = {
 	F(960000, P_XO, 10, 2, 5),
+	F(1000000, P_GPLL0, 16, 1, 50),
+	F(2000000, P_GPLL0, 16, 1, 25),
+	F(3000000, P_GPLL0, 16, 3, 50),
+	F(4000000, P_GPLL0, 16, 2, 25),
 	F(4800000, P_XO, 5, 0, 0),
+	F(5000000, P_GPLL0, 16, 1, 5),
+	F(6000000, P_GPLL0, 16, 3, 25),
+	F(6400000, P_GPLL0, 16, 16, 125),
+	F(7000000, P_GPLL0, 16, 7, 50),
+	F(8000000, P_GPLL0, 16, 4, 25),
+	F(9000000, P_GPLL0, 16, 9, 50),
 	F(9600000, P_XO, 2, 4, 5),
+	F(10000000, P_GPLL0, 16, 1, 5),
+	F(11000000, P_GPLL0, 16, 11, 50),
+	F(12000000, P_GPLL0, 16, 6, 25),
+	F(13000000, P_GPLL0, 16, 13, 50),
+	F(14000000, P_GPLL0, 16, 7, 25),
+	F(15000000, P_GPLL0, 16, 3, 10),
 	F(16000000, P_GPLL0, 10, 1, 5),
+
+	F(17000000, P_GPLL0, 16, 17, 50),
+	F(18000000, P_GPLL0, 16, 9, 25),
+	F(19000000, P_GPLL0, 16, 19, 50),
+
+	F(20000000, P_GPLL0, 16, 2, 5),
+	F(21000000, P_GPLL0, 16, 21, 50),
+	F(22000000, P_GPLL0, 16, 11, 25),
+	F(23000000, P_GPLL0, 16, 23, 50),
 	F(24000000, P_XO, 1, 0, 0),
 	F(25000000, P_GPLL0, 16, 1, 2),
+	F(26000000, P_GPLL0, 16, 13, 25),
+	F(27000000, P_GPLL0, 16, 27, 50),
+	F(28000000, P_GPLL0, 16, 14, 25),
+	F(29000000, P_GPLL0, 16, 29, 50),
+
+	F(30000000, P_GPLL0, 16, 3, 5),
+	F(31000000, P_GPLL0, 16, 31, 50),
+	F(32000000, P_GPLL0, 16, 16, 25),
+	F(33000000, P_GPLL0, 16, 33, 50),
+	F(34000000, P_GPLL0, 16, 17, 25),
+	F(35000000, P_GPLL0, 16, 7, 10),
+	F(36000000, P_GPLL0, 16, 18, 25),
+	F(37000000, P_GPLL0, 16, 37, 50),
+	F(38000000, P_GPLL0, 16, 19, 25),
+	F(39000000, P_GPLL0, 16, 39, 50),
+
+	F(40000000, P_GPLL0, 16, 4, 5),
+	F(41000000, P_GPLL0, 16, 41, 50),
+	F(42000000, P_GPLL0, 16, 21, 25),
+	F(43000000, P_GPLL0, 16, 43, 50),
+	F(44000000, P_GPLL0, 16, 22, 25),
+	F(45000000, P_GPLL0, 16, 9, 10),
+	F(46000000, P_GPLL0, 16, 23, 25),
+	F(47000000, P_GPLL0, 16, 47, 50),
+	F(48000000, P_GPLL0, 16, 24, 25),
+	F(49000000, P_GPLL0, 16, 49, 50),
 	F(50000000, P_GPLL0, 16, 0, 0),
 	{ }
 };
-- 
2.49.1

