Index: linux-6.6.88/drivers/net/phy/swconfig_leds.c
===================================================================
--- linux-6.6.88.orig/drivers/net/phy/swconfig_leds.c
+++ linux-6.6.88/drivers/net/phy/swconfig_leds.c
@@ -491,6 +491,9 @@ swconfig_led_work_func(struct work_struc
 						SWCONFIG_LED_PORT_SPEED_100;
 					break;
 				case SWITCH_PORT_SPEED_1000:
+				case SWITCH_PORT_SPEED_2500:
+				case SWITCH_PORT_SPEED_5000:
+				case SWITCH_PORT_SPEED_10000:
 					sw_trig->link_speed[i] =
 						SWCONFIG_LED_PORT_SPEED_1000;
 					break;
