Index: linux-6.6.88/net/netfilter/xt_FLOWOFFLOAD.c
===================================================================
--- linux-6.6.88.orig/net/netfilter/xt_FLOWOFFLOAD.c
+++ linux-6.6.88/net/netfilter/xt_FLOWOFFLOAD.c
@@ -62,7 +62,7 @@ xt_flowoffload_net_hook(void *priv, stru
 		proto = veth->h_vlan_encapsulated_proto;
 		break;
 	case htons(ETH_P_PPP_SES):
-		proto = nf_flow_pppoe_proto(skb);
+		proto = __nf_flow_pppoe_proto(skb);
 		break;
 	default:
 		proto = skb->protocol;
@@ -518,8 +518,7 @@ flowoffload_tg(struct sk_buff *skb, cons
 	if (!flow)
 		goto err_flow_alloc;
 
-	if (flow_offload_route_init(flow, &route) < 0)
-		goto err_flow_add;
+	flow_offload_route_init(flow, &route);
 
 	if (tcph) {
 		ct->proto.tcp.seen[0].flags |= IP_CT_TCP_FLAG_BE_LIBERAL;
