From cbed55017d1d496f9de63d350eb45124646661a2 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Wed, 25 Sep 2019 10:17:01 +0200
Subject: [PATCH] node: disable ASLR PIE

---
 node/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/node/Makefile b/node/Makefile
index ee8d6b98..e4644fb5 100644
--- a/node/Makefile
+++ b/node/Makefile
@@ -40,6 +40,7 @@ PKG_BUILD_DEPENDS:=!NODEJS_12:python/host NODEJS_12:python3/host NODEJS_12:node/
 PKG_BUILD_PARALLEL:=1
 PKG_INSTALL:=1
 PKG_USE_MIPS16:=0
+PKG_ASLR_PIE:=0
 
 OW_VER:=$(shell git --git-dir=$(TOPDIR)/.git describe --abbrev=0 --always --tag)
 
-- 
2.20.1

