From 886f2ec622abd5cad76cf3f60643750b1c15b123 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Wed, 25 Sep 2019 10:17:01 +0200
Subject: [PATCH] node: disable ASLR PIE

---
 node/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/node/Makefile b/node/Makefile
index d83356b..9cfe3f7 100644
--- a/node/Makefile
+++ b/node/Makefile
@@ -33,6 +33,7 @@ PKG_BUILD_DEPENDS:=python3/host
 PKG_BUILD_PARALLEL:=1
 PKG_INSTALL:=1
 PKG_USE_MIPS16:=0
+PKG_ASLR_PIE:=0
 
 include $(INCLUDE_DIR)/host-build.mk
 include $(INCLUDE_DIR)/package.mk
-- 
2.24.1

