From 4c0f06c394ceb95db7cc55e0dc6add07699c134b Mon Sep 17 00:00:00 2001
From: Jan Pavlinec <jan.pavlinec@nic.cz>
Date: Wed, 8 Apr 2020 12:59:05 +0200
Subject: [PATCH] meson: update to version 0.54.0

---
 devel/meson/Makefile | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/devel/meson/Makefile b/devel/meson/Makefile
index 1df0880..5bb4801 100644
--- a/devel/meson/Makefile
+++ b/devel/meson/Makefile
@@ -1,18 +1,18 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=meson
-PKG_VERSION:=0.53.2
+PKG_VERSION:=0.54.0
 PKG_RELEASE:=1
 
 PYPI_NAME:=meson
-PKG_HASH:=3e8f830f33184397c2eb0b651ec502adb63decb28978bdc84b3558d71284c21f
+PKG_HASH:=dde5726d778112acbd4a67bb3633ab2ee75d33d1e879a6283a7b4a44c3363c27
+
 
 PKG_MAINTAINER:=Andre Heider <a.heider@gmail.com>
 PKG_LICENSE:=Apache-2.0
 PKG_LICENSE_FILES:=COPYING
 
-HOST_BUILD_DEPENDS:=ninja/host
-
+HOST_BUILD_DEPENDS:=ninja/host python3/host
 include ../../lang/python/pypi.mk
 include $(INCLUDE_DIR)/host-build.mk
 include $(INCLUDE_DIR)/package.mk
-- 
2.17.1

