From c371b302a5796d9c240e6aefa4fabcead79a7fbf Mon Sep 17 00:00:00 2001
From: Jan Pavlinec <jan.pavlinec@nic.cz>
Date: Wed, 8 Apr 2020 12:59:24 +0200
Subject: [PATCH] meson: fix python python to python3

---
 devel/meson/meson.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/devel/meson/meson.mk b/devel/meson/meson.mk
index 16c6b16..a93ae77 100644
--- a/devel/meson/meson.mk
+++ b/devel/meson/meson.mk
@@ -37,7 +37,7 @@ MESON_VARS:=
 MESON_ARGS:=
 
 define Meson
-	$(2) $(STAGING_DIR_HOST)/bin/$(PYTHON) $(MESON_DIR)/meson.py $(1)
+	$(2) $(STAGING_DIR_HOSTPKG)/bin/python3 $(MESON_DIR)/meson.py $(1)
 endef
 
 define Meson/CreateNativeFile
-- 
2.17.1

