From db3b1c6e803f5b62bce3101ece31c4e1080e993e Mon Sep 17 00:00:00 2001
From: Jan Pavlinec <jan.pavlinec@nic.cz>
Date: Thu, 7 Feb 2019 17:38:07 +0100
Subject: [PATCH] protobuf-c: fix host build dependency

---
 libs/protobuf-c/Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/libs/protobuf-c/Makefile b/libs/protobuf-c/Makefile
index 71a0b04..c874843 100644
--- a/libs/protobuf-c/Makefile
+++ b/libs/protobuf-c/Makefile
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=libprotobuf-c
 PKG_VERSION:=v1.2.1
-PKG_RELEASE:=$(PKG_SOURCE_VERSION)
+PKG_RELEASE:=$(PKG_SOURCE_VERSION).1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
 PKG_MIRROR_HASH:=2ebe48454fe454d118cf952655a24477c4bed892cee7ae085dc56d05ac711a8a
@@ -19,6 +19,7 @@ PKG_SOURCE_PROTO:=git
 PKG_SOURCE_VERSION:=$(PKG_VERSION)
 
 PKG_BUILD_DEPENDS:=protobuf-c/host
+HOST_BUILD_DEPENDS:=protobuf/host
 
 PKG_INSTALL:=1
 PKG_FIXUP:=autoreconf
-- 
2.7.4

