From 4b6724e73e9c607613a5f25a659ecc8d99cdd964 Mon Sep 17 00:00:00 2001
From: Jan Pavlinec <jan.pavlinec@nic.cz>
Date: Fri, 8 Feb 2019 10:59:30 +0100
Subject: [PATCH] nut: disable web-cgi for now (missing libgd breaks build)

---
 net/nut/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/net/nut/Makefile b/net/nut/Makefile
index 61f15c2..226fc6e 100644
--- a/net/nut/Makefile
+++ b/net/nut/Makefile
@@ -19,7 +19,7 @@ PKG_LICENSE:=GPL-2.0+ GPL-3.0+ GPL-1.0+ Artistic-1.0-Perl
 PKG_LICENSE_FILES:=LICENSE-GPL2 LICENSE-GPL3 COPYING
 PKG_FIXUP:=autoreconf
 
-PKG_BUILD_DEPENDS:=libgd	
+PKG_BUILD_DEPENDS:=libgd
 
 PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
 PKG_INSTALL:=1
@@ -560,7 +560,7 @@ $(eval $(call BuildPackage,nut-upscmd))
 $(eval $(call BuildPackage,nut-upslog))
 $(eval $(call BuildPackage,nut-upsrw))
 $(eval $(call BuildPackage,nut-upssched))
-$(eval $(call BuildPackage,nut-web-cgi))
+#$(eval $(call BuildPackage,nut-web-cgi))
 $(eval $(call BuildPackage,nut-avahi-service))
 $(foreach d,$(filter-out $(SERIAL_DRIVERLIST_IGNORE),$(SERIAL_DRIVERLIST)),$(eval $(call BuildPackage,nut-driver-$(d))))
 $(foreach d,$(SNMP_DRIVERLIST),$(eval $(call BuildPackage,nut-driver-$(d))))
-- 
2.7.4

