From 293a114816847e21a535bdc5f654e245e888cb3d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Karel=20Ko=C4=8D=C3=AD?= <karel.koci@nic.cz>
Date: Tue, 12 Feb 2019 12:48:43 +0100
Subject: [PATCH] twisted: add Python3 variant
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Karel Kočí <karel.koci@nic.cz>
---
 lang/python/twisted/Makefile | 31 ++++++++++++++++++++++++++++++-
 1 file changed, 30 insertions(+), 1 deletion(-)

diff --git a/lang/python/twisted/Makefile b/lang/python/twisted/Makefile
index 35b78172b..7a318c10d 100644
--- a/lang/python/twisted/Makefile
+++ b/lang/python/twisted/Makefile
@@ -1,5 +1,5 @@
 #
-# Copyright (C) 2006-2011, 2015-2018 OpenWrt.org
+# Copyright (C) 2006-2011, 2015-2019 OpenWrt.org
 #
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
@@ -24,6 +24,7 @@ PKG_MAINTAINER:=Jeffery To <jeffery.to@gmail.com>
 
 include $(INCLUDE_DIR)/package.mk
 include ../python-package.mk
+include ../python3-package.mk
 
 PKG_UNPACK:=$(HOST_TAR) -C $(PKG_BUILD_DIR) --strip-components=1 -xjf $(DL_DIR)/$(PKG_SOURCE)
 
@@ -47,18 +48,46 @@ $(call Package/twisted/Default)
   VARIANT:=python
 endef
 
+define Package/python3-twisted
+$(call Package/twisted/Default)
+  TITLE:=Python3 networking engine
+  DEPENDS:= \
+      +PACKAGE_twisted:python3-light \
+      +PACKAGE_twisted:python3-attrs \
+      +PACKAGE_twisted:python3-automat \
+      +PACKAGE_twisted:python3-constantly \
+      +PACKAGE_twisted:python3-incremental \
+      +PACKAGE_twisted:python3-hyperlink \
+      +PACKAGE_twisted:python3-zope-interface
+  VARIANT:=python3
+endef
+
 define Package/twisted/description
 Twisted is a networking engine written in Python, supporting numerous
 protocols. It contains a web server, numerous chat clients, chat servers,
 mail servers, and more.
 endef
 
+define Package/python3-twisted/description
+$(call Package/twisted/description)
+.
+(Variant for Python3)
+endef
+
 define PyPackage/twisted/filespec
 +|/usr/bin
 +|$(PYTHON_PKG_DIR)
 -|$(PYTHON_PKG_DIR)/twisted/conch/scripts/tkconch.py
 endef
 
+define Py3Package/python3-twisted/filespec
+$(call PyPackage/twisted/filespec)
+endef
+
 $(eval $(call PyPackage,twisted))
 $(eval $(call BuildPackage,twisted))
 $(eval $(call BuildPackage,twisted-src))
+
+$(eval $(call Py3Package,python3-twisted))
+$(eval $(call BuildPackage,python3-twisted))
+$(eval $(call BuildPackage,python3-twisted-src))
-- 
2.20.1

