From 29044be0e33a00fbd22ddb0c4b47c203014457a5 Mon Sep 17 00:00:00 2001
From: Michal Vasilek <michal.vasilek@nic.cz>
Date: Thu, 23 Jun 2022 11:36:06 +0200
Subject: [PATCH 5/5] php7-pecl-redis: remove obsolete php7-mod-hash dep

---
 lang/php7-pecl-redis/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/lang/php7-pecl-redis/Makefile b/lang/php7-pecl-redis/Makefile
index 71f9ee9a6..97368f118 100644
--- a/lang/php7-pecl-redis/Makefile
+++ b/lang/php7-pecl-redis/Makefile
@@ -9,7 +9,7 @@ PECL_NAME:=redis
 PECL_LONGNAME:=PHP extension for interfacing with Redis
 
 PKG_VERSION:=4.3.0
-PKG_RELEASE:=1
+PKG_RELEASE:=2
 PKG_HASH:=c0f04cec349960a842b60920fb8a433656e2e494eaed6e663397d67102a51ba2
 
 PKG_NAME:=php7-pecl-redis
@@ -35,5 +35,5 @@ CONFIGURE_ARGS+= \
 	--disable-redis-igbinary \
 	--disable-redis-lzf
 
-$(eval $(call PECLPackage,$(PECL_NAME),$(PECL_LONGNAME),+php7-mod-hash +php7-mod-session,25))
+$(eval $(call PECLPackage,$(PECL_NAME),$(PECL_LONGNAME),+php7-mod-session,25))
 $(eval $(call BuildPackage,$(PKG_NAME)))
-- 
2.36.1

