From 8528aa137ad1cca17c5e55655fbe8f3b3e7b6fb8 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Fri, 25 Oct 2019 04:11:29 +0200
Subject: [PATCH] libseccomp: add scmp_sys_resolver package

Signed-off-by: Josef Schlehofer <pepe.schlehofer@gmail.com>
---
 libs/libseccomp/Makefile | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/libs/libseccomp/Makefile b/libs/libseccomp/Makefile
index a53b95ac7..48fed66b0 100644
--- a/libs/libseccomp/Makefile
+++ b/libs/libseccomp/Makefile
@@ -53,6 +53,12 @@ $(call Package/libseccomp/Default)
   TITLE+= (library)
 endef
 
+define Package/scmp_sys_resolver
+$(call Package/libseccomp/Default)
+  TITLE+= scmp_sys_resolver
+  DEPENDS+= libseccomp
+endef
+
 define Package/libseccomp/description
  This package contains the seccomp library.
 endef
@@ -75,4 +81,10 @@ define Package/libseccomp/install
 	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libseccomp.so.* $(1)/usr/lib/
 endef
 
+define Package/scmp_sys_resolver/install
+	$(INSTALL_DIR) $(1)/usr/bin
+	$(CP) $(PKG_INSTALL_DIR)/usr/bin/scmp_sys_resolver $(1)/usr/bin/
+endef
+
 $(eval $(call BuildPackage,libseccomp))
+$(eval $(call BuildPackage,scmp_sys_resolver))
-- 
2.20.1

