From eeb4329295d463c8e8c74008fed44348126536bb Mon Sep 17 00:00:00 2001
From: "W. Michael Petullo" <mike@flyn.org>
Date: Sun, 5 Aug 2018 20:21:52 -0400
Subject: [PATCH 1/2] luajit: add .hpp to InstallDev

Signed-off-by: W. Michael Petullo <mike@flyn.org>
---
 lang/luajit/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lang/luajit/Makefile b/lang/luajit/Makefile
index 0bbb292f1..94cf9094d 100644
--- a/lang/luajit/Makefile
+++ b/lang/luajit/Makefile
@@ -59,7 +59,7 @@ endef
 
 define Build/InstallDev
 	$(INSTALL_DIR) $(1)/usr/include/luajit-2.1
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/luajit-2.1/*.h $(1)/usr/include/luajit-2.1
+	$(CP) $(PKG_INSTALL_DIR)/usr/include/luajit-2.1/*.{h,hpp} $(1)/usr/include/luajit-2.1
 	$(INSTALL_DIR) $(1)/usr/lib
 	$(CP) $(PKG_INSTALL_DIR)/usr/lib/*.{a,so*} $(1)/usr/lib/
 	$(INSTALL_DIR) $(1)/usr/lib/pkgconfig
-- 
2.24.0

