From 46d64e226de89d78accabe92ed396402fa25d049 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <josef.schlehofer@nic.cz>
Date: Tue, 8 Dec 2020 20:38:26 +0100
Subject: [PATCH 2/2] rescue-image: fix path of gen_initramfs_list as it was
 moved

---
 hardware/rescue-image/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hardware/rescue-image/Makefile b/hardware/rescue-image/Makefile
index f8a9e28cb..20b5042b2 100644
--- a/hardware/rescue-image/Makefile
+++ b/hardware/rescue-image/Makefile
@@ -99,7 +99,7 @@ define Build/Compile
 	done
 	rm -rf "$(PKG_BUILD_DIR)/rootfs"/usr/lib/opkg
 	$(KERNEL_MAKE) $(PKG_JOBS) oldconfig $(KERNELNAME)
-	cd $(LINUX_DIR); ./scripts/gen_initramfs_list.sh \
+	cd $(LINUX_DIR); ./usr/gen_initramfs_list.sh \
 	-o "$(PKG_BUILD_DIR)"/kernel/linux-$(LINUX_VERSION)/arch/$(LINUX_KARCH)/boot/initrd.gz -u squash -g squash \
 	"$(PKG_BUILD_DIR)/rootfs" "$(PKG_BUILD_DIR)/rootfs.extra"
 	gzip -d "$(PKG_BUILD_DIR)"/kernel/linux-$(LINUX_VERSION)/arch/$(LINUX_KARCH)/boot/initrd.gz
-- 
2.25.1

