From b899a4127d20d6ac47d9ab6e5483e0f1ca9223d5 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Thu, 13 Jan 2022 13:38:37 +0100
Subject: [PATCH 3/4] nextcloud: update to version 21.0.7

---
 net/nextcloud/Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/net/nextcloud/Makefile b/net/nextcloud/Makefile
index 44ab53feee..44e59fc7e4 100644
--- a/net/nextcloud/Makefile
+++ b/net/nextcloud/Makefile
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=nextcloud
-PKG_VERSION:=20.0.4
-PKG_RELEASE:=4
+PKG_VERSION:=21.0.7
+PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
 PKG_SOURCE_URL:=https://download.nextcloud.com/server/releases/
-PKG_HASH:=269f1622e326f5d11e387d3861aad4e2b0e79334ae97eed5a7b3352ba7661420
+PKG_HASH:=599321596eb5deae5cea0ad1fddcd54822892abbbb5cfb51a31801862f1513b7
 
 PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)
 
-- 
2.34.1

