From ff502152eecebe8e4d571ad72fb87200c3e6d995 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Thu, 13 Oct 2022 22:34:50 +0200
Subject: [PATCH 1/2] kernel-version: add kernel 5.10 and kernel 5.15

---
 include/kernel-version.mk | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/include/kernel-version.mk b/include/kernel-version.mk
index 07e64d4edd..28ce352d1d 100644
--- a/include/kernel-version.mk
+++ b/include/kernel-version.mk
@@ -7,8 +7,12 @@ ifdef CONFIG_TESTING_KERNEL
 endif
 
 LINUX_VERSION-5.4 = .215
+LINUX_VERSION-5.10 = .147
+LINUX_VERSION-5.15 = .74
 
 LINUX_KERNEL_HASH-5.4.215 = 2d2bf7c3a25297f2b4686aa50a7cdaac68b52fc4c81b2cd3dddfe9f592b34ebd
+LINUX_KERNEL_HASH-5.10.147 = 85253abf097eb5013b988a400eb022eed0e0626e7e0a7daa3ab4b6bcabbced9a
+LINUX_KERNEL_HASH-5.15.74 = 2c1539a2f85b835c36c4a07c8270b52b0bec38fdda7339477d07f0c3af8c4265
 
 remove_uri_prefix=$(subst git://,,$(subst http://,,$(subst https://,,$(1))))
 sanitize_uri=$(call qstrip,$(subst @,_,$(subst :,_,$(subst .,_,$(subst -,_,$(subst /,_,$(1)))))))
-- 
2.34.1

