From 56079e545ce4734ee78ac5f91923e95503e996dd Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Wed, 27 Jul 2022 13:32:49 +0200
Subject: [PATCH] mvebu/cortexa9: enable earlyprintk

---
 target/linux/mvebu/cortexa9/config-5.15 | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/target/linux/mvebu/cortexa9/config-5.15 b/target/linux/mvebu/cortexa9/config-5.15
index b737c44d42..b1f2452d52 100644
--- a/target/linux/mvebu/cortexa9/config-5.15
+++ b/target/linux/mvebu/cortexa9/config-5.15
@@ -1,2 +1,7 @@
 CONFIG_PHY_MVEBU_A38X_COMPHY=y
 CONFIG_RTC_DRV_MV=y
+
+#Earlyprintk (https://gitlab.nic.cz/turris/os/build/-/issues/347)
+CONFIG_CMDLINE="earlyprintk console=ttyS0,115200"
+CONFIG_CMDLINE_FROM_BOOTLOADER=y
+CONFIG_DEBUG_MVEBU_UART0_ALTERNATE=y
-- 
2.34.1

