From 5943e623fd01f7c8ad74f0b6f877cc0108b4792f Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Sat, 27 Aug 2022 11:46:47 +0200
Subject: [PATCH] enable CONFIG_MTD_NAND_FSL_ELBC

---
 target/linux/mpc85xx/p2020/config-default | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/target/linux/mpc85xx/p2020/config-default b/target/linux/mpc85xx/p2020/config-default
index ef32280dac..5093f2bc6e 100644
--- a/target/linux/mpc85xx/p2020/config-default
+++ b/target/linux/mpc85xx/p2020/config-default
@@ -7,6 +7,10 @@ CONFIG_GENERIC_TBSYNC=y
 CONFIG_IRQ_DOMAIN_HIERARCHY=y
 CONFIG_IRQ_DOMAIN_NOMAP=y
 CONFIG_LOCK_SPIN_ON_OWNER=y
+CONFIG_MTD_CFI=y
+CONFIG_MTD_NAND_BCH=y
+CONFIG_MTD_NAND_ECC_BCH=y
+CONFIG_MTD_NAND_FSL_ELBC=y
 CONFIG_MTD_PHYSMAP=y
 CONFIG_MUTEX_SPIN_ON_OWNER=y
 CONFIG_NET_FLOW_LIMIT=y
-- 
2.34.1

