From b7f455433fe186a3651037113f7eb3e10c7a5674 Mon Sep 17 00:00:00 2001
From: Jan Pavlinec <jan.pavlinec@nic.cz>
Date: Thu, 25 Mar 2021 15:21:59 +0100
Subject: [PATCH] php7: bump PKG_RELEASE because of memory_limi change

Signed-off-by: Jan Pavlinec <jan.pavlinec@nic.cz>
---
 lang/php7/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lang/php7/Makefile b/lang/php7/Makefile
index 44f7d59..2f5b8f4 100644
--- a/lang/php7/Makefile
+++ b/lang/php7/Makefile
@@ -7,7 +7,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=php
 PKG_VERSION:=7.2.34
-PKG_RELEASE:=3
+PKG_RELEASE:=4
 
 PKG_MAINTAINER:=Michael Heimpold <mhei@heimpold.de>
 
-- 
2.17.1

