--- a/mk/obj.mk	2025-08-05 11:56:35.463273268 +0200
+++ b/mk/obj.mk	2025-08-05 11:56:39.043262462 +0200
@@ -442,7 +442,7 @@
                src/ref/ref_uci.o src/hsl/hsl_dev.o src/hsl/hsl_api.o \
                src/hsl/hsl_port_prop.o src/util/util.o src/sal/sd/sd.o \
                src/sal/sd/linux/uk_interface/sw_api_ks_ioctl.o \
-               src/api/api_access.o
+               src/api/api_access.o src/hsl/mht/mht_interface_ctrl.o
 OBJ-ISISC   += src/hsl/isisc/isisc_reg_access.o
 OBJ-HPPE    += src/hsl/hppe/hppe_global.o src/hsl/hppe/hppe_reg_access.o
 OBJ-APPE    += src/hsl/appe/appe_counter.o src/hsl/appe/appe_global.o \
