--- a/src/init/ssdk_dts.c	2025-08-05 11:12:29.242270695 +0200
+++ b/src/init/ssdk_dts.c	2025-08-05 11:14:15.986945245 +0200
@@ -734,7 +734,7 @@
 		SSDK_ERROR("i2c adpt was not found\n");
 		return NULL;
 	}
-	mdio_i2c = mdio_i2c_alloc(&(i2c_adpt->dev), i2c_adpt, MIDO_I2C_QCOM);
+	mdio_i2c = mdio_i2c_alloc(&(i2c_adpt->dev), i2c_adpt, MDIO_I2C_C45);
 	if (!mdio_i2c) {
 		SSDK_ERROR("mdio_i2c bus alloc failed\n");
 		return NULL;
