From d3685151672fd4977f809ee285f4effe673187a5 Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <michal.hrusecky@turris.com>
Date: Mon, 28 Jul 2025 23:30:50 +0200
Subject: [PATCH] kernel/netfilter: Add pptp dependency

---
 package/kernel/linux/modules/netfilter.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/package/kernel/linux/modules/netfilter.mk b/package/kernel/linux/modules/netfilter.mk
index 96531b1e08..60a03e06ca 100644
--- a/package/kernel/linux/modules/netfilter.mk
+++ b/package/kernel/linux/modules/netfilter.mk
@@ -102,6 +102,7 @@ $(eval $(call KernelPackage,ipt-core))
 define KernelPackage/nf-conntrack
   SUBMENU:=$(NF_MENU)
   TITLE:=Netfilter connection tracking
+  DEPENDS:=+kmod-pptp
   KCONFIG:= \
         CONFIG_NETFILTER=y \
         CONFIG_NETFILTER_ADVANCED=y \
-- 
2.50.1

