From 47b910e5eac5dc0e0d8d3f17d05a441a1a63af3a Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Sun, 4 Dec 2022 10:44:44 +0100
Subject: [PATCH] include/kernels: add dedicated file for kernel 5.15 version
 5.15

Signed-off-by: Josef Schlehofer <pepe.schlehofer@gmail.com>
---
 include/kernel-5.15 | 2 ++
 2 files changed, 4 insertions(+)
 create mode 100644 include/kernel-5.15

diff --git a/include/kernel-5.10 b/include/kernel-5.10
new file mode 100644
index 0000000000..16040014ff
--- /dev/null
+++ b/include/kernel-5.10
@@ -0,0 +1,2 @@
+LINUX_VERSION-5.10 = .160
+LINUX_KERNEL_HASH-5.10.160 = 30d5076acae863941045880c4c5c5109d26a54a932168fa1324237e8aeaa840b
diff --git a/include/kernel-5.15 b/include/kernel-5.15
new file mode 100644
index 0000000000..a95ca029de
--- /dev/null
+++ b/include/kernel-5.15
@@ -0,0 +1,2 @@
+LINUX_VERSION-5.15 = .84
+LINUX_KERNEL_HASH-5.15.84 = 318dc30cb059c2e35b59652b166b39804bb3a941f11878aae6119019a04b8217
-- 
2.38.1

