From 3765f3e97fa12b3fed4b987c0911772e39086902 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Sun, 4 Dec 2022 10:44:44 +0100
Subject: [PATCH] include/kernels: add dedicated file for kernel 5.15 version
 5.15

Signed-off-by: Josef Schlehofer <pepe.schlehofer@gmail.com>
---
 include/kernel-5.15 | 2 ++
 1 file changed, 2 insertions(+)
 create mode 100644 include/kernel-5.15

diff --git a/include/kernel-5.15 b/include/kernel-5.15
new file mode 100644
index 0000000000..4c18bc35c2
--- /dev/null
+++ b/include/kernel-5.15
@@ -0,0 +1,2 @@
+LINUX_VERSION-5.15 = .142
+LINUX_KERNEL_HASH-5.15.142 = 8d76c95277dc5ab0a6cd0069432af2ceb759d0ac2b6f5401330d390196095676
-- 
2.40.1

