From 66eb8ba04fa18a6ebcea67ebe43b2a9e496ea575 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Sun, 4 Dec 2022 10:44:44 +0100
Subject: [PATCH] include/kernels: add dedicated file for kernel 5.15 version
 5.15

Signed-off-by: Josef Schlehofer <pepe.schlehofer@gmail.com>
---
 include/kernel-5.15 | 2 ++
 2 files changed, 4 insertions(+)
 create mode 100644 include/kernel-5.15

diff --git a/include/kernel-5.15 b/include/kernel-5.15
new file mode 100644
index 0000000000..ab1500f4b2
--- /dev/null
+++ b/include/kernel-5.15
@@ -0,0 +1,2 @@
+LINUX_VERSION-5.15 = .94
+LINUX_KERNEL_HASH-5.15.94 = da9270dbe64ddf1db13c70470957ff6796eb996d867bb4aed7d14a70e1c65a72
-- 
2.39.1

