From 5a41841d97e2a2d0d05e7ade9a049eafae756e3a Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Sun, 4 Dec 2022 10:44:44 +0100
Subject: [PATCH] include/kernels: add dedicated file for kernel 5.15 version
 5.15

Signed-off-by: Josef Schlehofer <pepe.schlehofer@gmail.com>
---
 include/kernel-5.15 | 2 ++
 2 files changed, 4 insertions(+)
 create mode 100644 include/kernel-5.15

diff --git a/include/kernel-5.15 b/include/kernel-5.15
new file mode 100644
index 0000000000..5c90c189c4
--- /dev/null
+++ b/include/kernel-5.15
@@ -0,0 +1,2 @@
+LINUX_VERSION-5.15 = .96
+LINUX_KERNEL_HASH-5.15.96 = 348d974c143fdef8517ec703fdaa24bade12a49047848be92cb9e3253b19ef98
-- 
2.39.2

