From 52fc185eba49b5cf5a282b89a29ae0c29eb10a84 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marek=20Beh=C3=BAn?= <marek.behun@nic.cz>
Date: Thu, 24 Sep 2020 00:28:51 +0200
Subject: [PATCH] mvebu: add support for Turris SFP module

---
 target/linux/mvebu/config-5.15                 |   1 +

diff --git a/target/linux/mvebu/config-5.15 b/target/linux/mvebu/config-5.15
index ea73bafe23..5c4479357a 100644
--- a/target/linux/mvebu/config-5.15
+++ b/target/linux/mvebu/config-5.15
@@ -241,6 +243,7 @@ CONFIG_MACH_MVEBU_V7=y
 CONFIG_MAGIC_SYSRQ=y
 CONFIG_MANGLE_BOOTARGS=y
 CONFIG_MARVELL_PHY=y
+CONFIG_MARVELL_10G_PHY=y
 CONFIG_MDIO_BUS=y
 CONFIG_MDIO_DEVICE=y
 CONFIG_MDIO_DEVRES=y
