From fb23cd1f23777ce7985ae108b8538ae618a4fc20 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Fri, 29 Oct 2021 16:05:45 +0200
Subject: [PATCH] fix compilation in env/flags.c

---
 env/flags.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/env/flags.c
+++ b/env/flags.c
@@ -6,7 +6,7 @@
 
 #include <env.h>
 #include <linux/string.h>
-#include <linux/ctype.h>
+#include <ctype.h>
 
 #ifdef USE_HOSTCC /* Eliminate "ANSI does not permit..." warnings */
 #include <stdint.h>
