From 499d48b393eb582e591e7b4af181b79732647a76 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Wed, 4 May 2022 17:41:03 +0200
Subject: [PATCH] mpc85xx: add drivers for PHY on Turris 1.x routers

---
 target/linux/mpc85xx/config-4.14 | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/target/linux/mpc85xx/config-4.14 b/target/linux/mpc85xx/config-4.14
index c42ca58399..44ff16b6e2 100644
--- a/target/linux/mpc85xx/config-4.14
+++ b/target/linux/mpc85xx/config-4.14
@@ -378,6 +378,8 @@ CONFIG_SRCU=y
 CONFIG_SWCONFIG=y
 CONFIG_SWIOTLB=y
 CONFIG_SWPHY=y
+CONFIG_MICREL_PHY=y
+CONFIG_AT803X_PHY=y
 CONFIG_SYSCTL_EXCEPTION_TRACE=y
 CONFIG_TASK_SIZE=0xc0000000
 CONFIG_THREAD_SHIFT=13
-- 
2.34.1

