From f9cb0329365496873097f69356e7f87aa0c85205 Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <michal.hrusecky@turris.com>
Date: Mon, 7 Jul 2025 16:04:15 +0200
Subject: [PATCH 1/2] u-boot: Add python 3 dependency

---
 package/boot/uboot-tools/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package/boot/uboot-tools/Makefile b/package/boot/uboot-tools/Makefile
index 6b1b84de7f..05de9eb7a0 100644
--- a/package/boot/uboot-tools/Makefile
+++ b/package/boot/uboot-tools/Makefile
@@ -14,7 +14,7 @@ PKG_HASH:=cdef7d507c93f1bbd9f015ea9bc21fa074268481405501945abc6f854d5b686f
 PKG_SOURCE_SUBDIR:=$(PKG_DISTNAME)-$(PKG_VERSION)
 PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_DISTNAME)-$(PKG_VERSION)
 
-PKG_BUILD_DEPENDS:=fstools
+PKG_BUILD_DEPENDS:=fstools python3/host
 
 PKG_LICENSE:=GPL-2.0 GPL-2.0+
 PKG_LICENSE_FILES:=Licenses/README
-- 
2.50.0

