From dcf6250f15d59ba27616cbc7313870fd6452aa8a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marek=20Beh=C3=BAn?= <marek.behun@nic.cz>
Date: Thu, 19 Oct 2023 11:07:16 +0200
Subject: [PATCH] kernel: bump release version to 2
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Marek Behún <marek.behun@nic.cz>
---
 include/kernel-version.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/kernel-version.mk b/include/kernel-version.mk
index 1edb01df27..4b1c852a08 100644
--- a/include/kernel-version.mk
+++ b/include/kernel-version.mk
@@ -1,6 +1,6 @@
 
 # Use the default kernel version if the Makefile doesn't override it
-LINUX_RELEASE?=1
+LINUX_RELEASE?=2
 
 ifdef CONFIG_TESTING_KERNEL
   KERNEL_PATCHVER:=$(KERNEL_TESTING_PATCHVER)
-- 
2.41.0

