From b1adf14b1ca00fd21d8bb1b3ca8dfa4119b902ba Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <Michal@Hrusecky.net>
Date: Tue, 24 Jul 2018 22:58:13 +0200
Subject: [PATCH] busybox: Make sure we have /usr/sbin before installing files
 inside

---
 package/utils/busybox/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/package/utils/busybox/Makefile b/package/utils/busybox/Makefile
index ff8cdde..79261b9 100644
--- a/package/utils/busybox/Makefile
+++ b/package/utils/busybox/Makefile
@@ -145,6 +145,7 @@ ifneq ($(CONFIG_BUSYBOX_$(BUSYBOX_SYM)_CROND),)
 endif
 ifneq ($(CONFIG_BUSYBOX_$(BUSYBOX_SYM)_NTPD),)
 	$(INSTALL_BIN) ./files/sysntpd $(1)/etc/init.d/sysntpd
+	$(INSTALL_DIR) $(1)/usr/sbin
 	$(INSTALL_BIN) ./files/ntpd-hotplug $(1)/usr/sbin/ntpd-hotplug
 endif
 	-rm -rf $(1)/lib64
-- 
2.18.0

