From 4e6eaecf1a5eab735472e922470df38a6c6300c5 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Karel=20Ko=C4=8D=C3=AD?= <karel.koci@nic.cz>
Date: Mon, 2 Jul 2018 12:24:40 +0200
Subject: [PATCH] curl: add host build

---
 package/network/utils/curl/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/package/network/utils/curl/Makefile b/package/network/utils/curl/Makefile
index b4e3fd9..f866809 100644
--- a/package/network/utils/curl/Makefile
+++ b/package/network/utils/curl/Makefile
@@ -67,6 +67,7 @@ PKG_CONFIG_DEPENDS:= \
   CONFIG_LIBCURL_NTLM
 
 include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/host-build.mk
 
 define Package/curl/Default
   SECTION:=net
@@ -181,3 +182,4 @@ endef
 
 $(eval $(call BuildPackage,curl))
 $(eval $(call BuildPackage,libcurl))
+$(eval $(call HostBuild))
-- 
2.18.0

