From a429736c56af069b6511d623679f2bb1939515b4 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Thu, 2 Apr 2020 16:05:45 +0200
Subject: [PATCH] dvb: add support for TDA18250 tuner

---
 package/kernel/linux/modules/dvb.mk | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)

diff --git a/package/kernel/linux/modules/dvb.mk b/package/kernel/linux/modules/dvb.mk
index 3db464e10e..4f8e7df3e1 100644
--- a/package/kernel/linux/modules/dvb.mk
+++ b/package/kernel/linux/modules/dvb.mk
@@ -807,6 +807,22 @@ define MediaTuner
   AUTOLOAD:=$(call AutoProbe,$1)
 endef
 
+## START: Added in kernel 4.16
+
+define KernelPackage/media-tuner-tda18250
+  TITLE:=NXP TDA18250 silicon tuner
+  $(call MediaTuner,tda18250,CONFIG_MEDIA_TUNER_TDA18250)
+  DEPENDS+=+kmod-regmap-i2c +kmod-video-core
+endef
+
+define KernelPackage/media-tuner-tda18250/description
+ NXP TDA18250 silicon tuner driver.
+endef
+
+$(eval $(call KernelPackage,media-tuner-tda18250))
+
+## End: Added in kernel 4.16
+
 define KernelPackage/media-tuner-e4000
   TITLE:=Elonics E4000 silicon tuner
   $(call MediaTuner,e4000,CONFIG_MEDIA_TUNER_E4000)
-- 
2.26.0

