From aa5764c93f273457a6c5467a04cc2b29cc2504ff Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Karel=20Ko=C4=8D=C3=AD?= <cynerd@email.cz>
Date: Sun, 10 Mar 2019 10:50:53 +0100
Subject: [PATCH] libevent2: add host build
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Karel Kočí <cynerd@email.cz>
---
 package/libs/libevent2/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/package/libs/libevent2/Makefile b/package/libs/libevent2/Makefile
index 5d56f37c51..5d3f150e2b 100644
--- a/package/libs/libevent2/Makefile
+++ b/package/libs/libevent2/Makefile
@@ -23,6 +23,7 @@ PKG_FIXUP:=autoreconf
 PKG_INSTALL:=1
 PKG_BUILD_PARALLEL:=1
 
+include $(INCLUDE_DIR)/host-build.mk
 include $(INCLUDE_DIR)/package.mk
 
 define Package/libevent2/Default
@@ -151,6 +152,7 @@ define Package/libevent2-pthreads/install
 	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libevent_pthreads-2.0.so.* $(1)/usr/lib/
 endef
 
+$(eval $(call HostBuild))
 $(eval $(call BuildPackage,libevent2))
 $(eval $(call BuildPackage,libevent2-core))
 $(eval $(call BuildPackage,libevent2-extra))
-- 
2.21.0

