From 4383917d6ab17810286e8cfae7c035363abb03d4 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Karel=20Ko=C4=8D=C3=AD?= <cynerd@email.cz>
Date: Sun, 10 Mar 2019 10:50:53 +0100
Subject: [PATCH] libevent2: add host build
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Karel Kočí <cynerd@email.cz>
---
 package/libs/libevent2/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/package/libs/libevent2/Makefile b/package/libs/libevent2/Makefile
index 687f1b93a7..f4637b4dee 100644
--- a/package/libs/libevent2/Makefile
+++ b/package/libs/libevent2/Makefile
@@ -24,6 +24,7 @@ PKG_CONFIG_DEPENDS:= \
 	CONFIG_PACKAGE_libevent2-openssl \
 	CONFIG_PACKAGE_libevent2-mbedtls

+include $(INCLUDE_DIR)/host-build.mk
 include $(INCLUDE_DIR)/package.mk
 include $(INCLUDE_DIR)/cmake.mk

@@ -158,6 +159,7 @@ define Package/libevent2-pthreads/install
 	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libevent_pthreads-2.1.so.* $(1)/usr/lib/
 endef

+$(eval $(call HostBuild))
 $(eval $(call BuildPackage,libevent2))
 $(eval $(call BuildPackage,libevent2-core))
 $(eval $(call BuildPackage,libevent2-extra))
--
2.20.1
