From 54d784edaa19c551318cd934b670d315b9a34b6f Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <michal.hrusecky@nic.cz>
Date: Wed, 20 Mar 2019 16:42:28 +0100
Subject: [PATCH] u-boot tools: Add mkimage tool

---
 package/boot/uboot-tools/Makefile | 21 ++++++++++++++++++++-
 1 file changed, 20 insertions(+), 1 deletion(-)

diff --git a/package/boot/uboot-tools/Makefile b/package/boot/uboot-tools/Makefile
index d58b0d1118..6b1b84de7f 100644
--- a/package/boot/uboot-tools/Makefile
+++ b/package/boot/uboot-tools/Makefile
@@ -38,6 +38,19 @@ define Package/dumpimage/description
 	position to outfile.
 endef
 
+define Package/mkimage
+	SECTION:=utils
+	CATEGORY:=Utilities
+	SUBMENU:=Boot Loaders
+	TITLE:=create U-Boot bootloader images
+	URL:=http://www.denx.de/wiki/U-Boot
+	DEPENDS:=+libopenssl @OPENSSL_ENGINE
+endef
+
+define Package/uboot-mkimage/description
+	This package includes tools to create U-Boot bootloader images.
+endef
+
 define Package/fit-check-sign
 	SECTION:=utils
 	CATEGORY:=Utilities
@@ -86,7 +99,7 @@ define Build/Compile
 ifneq ($(CONFIG_PACKAGE_uboot-envtools),)
 	$(call Build/Compile/Default,envtools)
 endif
-ifneq ($(CONFIG_PACKAGE_dumpimage)$(CONFIG_PACKAGE_fit-check-sign),)
+ifneq ($(CONFIG_PACKAGE_dumpimage)$(CONFIG_PACKAGE_fit-check-sign)$(CONFIG_PACKAGE_mkimage),)
 	$(call Build/Compile/Default,cross_tools)
 endif
 
@@ -97,6 +110,11 @@ define Package/dumpimage/install
 	$(INSTALL_BIN) $(PKG_BUILD_DIR)/tools/dumpimage $(1)/usr/bin
 endef
 
+define Package/mkimage/install
+	$(INSTALL_DIR) $(1)/usr/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/tools/mkimage $(1)/usr/bin
+endef
+
 define Package/fit-check-sign/install
 	$(INSTALL_DIR) $(1)/usr/bin
 	$(INSTALL_BIN) $(PKG_BUILD_DIR)/tools/fit_check_sign $(1)/usr/bin
@@ -129,5 +147,6 @@ define Package/uboot-envtools/install
 endef
 
 $(eval $(call BuildPackage,dumpimage))
+$(eval $(call BuildPackage,mkimage))
 $(eval $(call BuildPackage,fit-check-sign))
 $(eval $(call BuildPackage,uboot-envtools))
-- 
2.50.0

