From c8d738cc35e81276ca51264dbd877aa9741acab8 Mon Sep 17 00:00:00 2001
From: Jan Pavlinec <jan.pavlinec@nic.cz>
Date: Fri, 23 Apr 2021 11:39:38 +0200
Subject: [PATCH] uboot-tools: disable EFI boot for turris1x

---
 package/boot/uboot-tools/Makefile | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/package/boot/uboot-tools/Makefile b/package/boot/uboot-tools/Makefile
index 284bec1624..4252c3ec3b 100644
--- a/package/boot/uboot-tools/Makefile
+++ b/package/boot/uboot-tools/Makefile
@@ -81,6 +81,9 @@ define Build/Compile
 	echo 'CONFIG_FIT=y' >> $(PKG_BUILD_DIR)/.config
 	echo 'CONFIG_MULTI_DTB_FIT=n' >> $(PKG_BUILD_DIR)/.config
 	sed -i 's|^CONFIG_TOOLS_MKEFICAPSULE=y|CONFIG_TOOLS_MKEFICAPSULE=n|' $(PKG_BUILD_DIR)/.config
+ifeq ($(CONFIG_TARGET_mpc85xx_p2020_DEVICE_cznic_turris1x),y)
+	sed -i 's|^\(CONFIG_EFI_LOADER=y\)|CONFIG_EFI_LOADER=n|' $(PKG_BUILD_DIR)/.config
+endif
 	$(MAKE) -C $(PKG_BUILD_DIR) \
 		CROSS_COMPILE="$(TARGET_CROSS)" \
 		TARGET_CFLAGS="$(TARGET_CFLAGS)" \
-- 
2.40.0

