From bcf41acdb2a94c3667226db8e3fdfcfab27470b9 Mon Sep 17 00:00:00 2001
From: Jan Pavlinec <jan.pavlinec@nic.cz>
Date: Fri, 23 Apr 2021 11:39:38 +0200
Subject: [PATCH] uboot-tools: disable EFI boot for turris1x

---
 package/boot/uboot-tools/Makefile | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/package/boot/uboot-tools/Makefile b/package/boot/uboot-tools/Makefile
index 1e2c35178c..467d0c9b86 100644
--- a/package/boot/uboot-tools/Makefile
+++ b/package/boot/uboot-tools/Makefile
@@ -80,6 +80,9 @@ define Build/Compile
 	mv $(PKG_BUILD_DIR)/tools/env/fw_printenv $(PKG_BUILD_DIR)
 	sed -i 's|^\(CONFIG_RSA.*\)=y|# \1 is not set|' $(PKG_BUILD_DIR)/.config
 	sed -i 's|^\(CONFIG_FIT\)=y|# \1 is not set|' $(PKG_BUILD_DIR)/.config
+ifeq ($(CONFIG_TARGET_mpc85xx_p2020_DEVICE_turris1x),y)
+	sed -i 's|^\(CONFIG_EFI_LOADER=y\)|CONFIG_EFI_LOADER=n|' $(PKG_BUILD_DIR)/.config
+endif
 	$(MAKE) -C $(PKG_BUILD_DIR) \
 		CROSS_COMPILE="$(TARGET_CROSS)" \
 		TARGET_CFLAGS="$(TARGET_CFLAGS)" \
--
2.17.1

