From 29546617a3735bfb20e261a842ee5051c659ca33 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Karel=20Ko=C4=8D=C3=AD?= <karel.koci@nic.cz>
Date: Tue, 17 Dec 2019 16:06:41 +0100
Subject: [PATCH] target/linux/mpc85xx: add missing option

---
 target/linux/mpc85xx/config-4.14 | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/target/linux/mpc85xx/config-4.14 b/target/linux/mpc85xx/config-4.14
index 3fb04ca..b7c4815 100644
--- a/target/linux/mpc85xx/config-4.14
+++ b/target/linux/mpc85xx/config-4.14
@@ -404,3 +404,5 @@ CONFIG_WATCHDOG_CORE=y
 # CONFIG_XPS_USB_HCD_XILINX is not set
 CONFIG_XZ_DEC_BCJ=y
 CONFIG_XZ_DEC_POWERPC=y
+# CONFIG_FB_FSL_DIU is not set
+# CONFIG_VGA_CONSOLE is not set
-- 
2.24.1

