From 8d5762aede14c0368e52d9a5624b6d11c5752f31 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Wed, 6 Apr 2022 14:35:59 +0200
Subject: [PATCH] kernel: add missing symbol for LTS 4.14

---
 target/linux/generic/config-4.14 | 1 +
 1 file changed, 1 insertion(+)

diff --git a/target/linux/generic/config-4.14 b/target/linux/generic/config-4.14
index 217754caa6..2027a0f83d 100644
--- a/target/linux/generic/config-4.14
+++ b/target/linux/generic/config-4.14
@@ -2653,6 +2653,7 @@ CONFIG_MESSAGE_LOGLEVEL_DEFAULT=4
 # CONFIG_MICROCHIP_PHY is not set
 # CONFIG_MICROSEMI_PHY is not set
 # CONFIG_MIGRATION is not set
+CONFIG_MITIGATE_SPECTRE_BRANCH_HISTORY=y
 CONFIG_MII=y
 # CONFIG_MIKROTIK_RB532 is not set
 # CONFIG_MIKROTIK is not set
-- 
2.32.0

