From f7512dcf9406e017a7bfdd19b87e552fc482339e Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Wed, 27 Jul 2022 14:04:21 +0200
Subject: [PATCH] mvebu: cortexa53: enable earlyprintk

---
 target/linux/mvebu/cortexa53/config-5.15 | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/target/linux/mvebu/cortexa53/config-5.15 b/target/linux/mvebu/cortexa53/config-5.15
index 955570cd54..c29a13ce73 100644
--- a/target/linux/mvebu/cortexa53/config-5.15
+++ b/target/linux/mvebu/cortexa53/config-5.15
@@ -98,3 +98,6 @@ CONFIG_CRYPTO_SHA256_ARM64=y
 CONFIG_CRYPTO_SHA2_ARM64_CE=y
 CONFIG_CRYPTO_SHA512_ARM64=y
 CONFIG_CRYPTO_CHACHA20_NEON=y
+CONFIG_CMDLINE="earlycon=ar3700_uart,0xd0012000 console=ttyMV0,115200"
+CONFIG_CMDLINE_FROM_BOOTLOADER=y
+CONFIG_SERIAL_EARLYCON=y
-- 
2.34.1

