From baf90394061d9ee38dc873d5f986cd3aecb01e2c Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Thu, 13 Oct 2022 15:09:45 +0200
Subject: [PATCH] config-5.15: add missing kernel options

Signed-off-by: Josef Schlehofer <pepe.schlehofer@gmail.com>
---
 target/linux/generic/config-5.15 | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/target/linux/generic/config-5.15 b/target/linux/generic/config-5.15
index 2950ea1a6b..054339882d 100644
--- a/target/linux/generic/config-5.15
+++ b/target/linux/generic/config-5.15
@@ -2401,6 +2401,11 @@ CONFIG_HIGH_RES_TIMERS=y
 # CONFIG_HOSTAP_PLX is not set
 # CONFIG_HOTPLUG_CPU is not set
 # CONFIG_HOTPLUG_PCI is not set
+# CONFIG_HOTPLUG_PCI_ACPI_IBM is not set
+# CONFIG_HOTPLUG_PCI_COMPAQ is not set
+# CONFIG_HOTPLUG_PCI_CPCI is not set
+# CONFIG_HOTPLUG_PCI_IBM is not set
+# CONFIG_HOTPLUG_PCI_SHPC is not set
 # CONFIG_HP03 is not set
 # CONFIG_HP100 is not set
 # CONFIG_HP206C is not set
-- 
2.34.1

