From 1070792131137dec8f7a25f432b7e76c0e62f6c7 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Mon, 12 Dec 2022 15:14:05 +0100
Subject: [PATCH] kernel-version: bump version to 2 to ensure mac80211 update

---
 include/kernel-version.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/kernel-version.mk b/include/kernel-version.mk
index 1edb01df27..4b1c852a08 100644
--- a/include/kernel-version.mk
+++ b/include/kernel-version.mk
@@ -1,6 +1,6 @@
 
 # Use the default kernel version if the Makefile doesn't override it
-LINUX_RELEASE?=1
+LINUX_RELEASE?=3
 
 ifdef CONFIG_TESTING_KERNEL
   KERNEL_PATCHVER:=$(KERNEL_TESTING_PATCHVER)
-- 
2.34.1

