From f721a6c66bc3d2a65a36b36f2e73104e3d3c17b0 Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <michal.hrusecky@turris.com>
Date: Thu, 20 Nov 2025 21:47:19 +0100
Subject: [PATCH] download.mk: Add possibility to pass git options

---
 include/download.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/download.mk b/include/download.mk
index 518a14e035..da7d8eb425 100644
--- a/include/download.mk
+++ b/include/download.mk
@@ -331,6 +331,7 @@ define Download/default
   SOURCE_VERSION:=$(PKG_SOURCE_VERSION)
   $(if $(PKG_MD5SUM),MD5SUM:=$(PKG_MD5SUM))
   $(if $(PKG_HASH),HASH:=$(PKG_HASH))
+  $(if $(PKG_GIT_OPTS),OPTS:=$(PKG_GIT_OPTS))
 endef
 
 define Download
-- 
2.51.1

