From e638dae8d603a9c84f7c55ed448d03e0b68d87b2 Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <michal.hrusecky@turris.com>
Date: Tue, 17 Mar 2026 15:56:48 +0100
Subject: [PATCH] nat46: Install devel files

---
 package/kernel/nat46/Makefile | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/package/kernel/nat46/Makefile b/package/kernel/nat46/Makefile
index 296ef5a058..fc511aa354 100644
--- a/package/kernel/nat46/Makefile
+++ b/package/kernel/nat46/Makefile
@@ -32,4 +32,9 @@ define Build/Compile
 		modules
 endef
 
+define Build/InstallDev/nat46
+	$(INSTALL_DIR) $(1)/usr/include/nat46
+	$(CP) $(PKG_BUILD_DIR)/nat46/modules/*.h $(1)/usr/include/nat46/
+endef
+
 $(eval $(call KernelPackage,nat46))
-- 
2.53.0

