From 0da12bf45a27ceecf1bbf5575f0b783cea2d6327 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Fri, 13 May 2022 19:20:28 +0200
Subject: [PATCH] kernel: update to version 4.14.280

---
 include/kernel-version.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/kernel-version.mk b/include/kernel-version.mk
index 1d97de0ec0..4a0c1c75ce 100644
--- a/include/kernel-version.mk
+++ b/include/kernel-version.mk
@@ -6,9 +6,9 @@ ifdef CONFIG_TESTING_KERNEL
   KERNEL_PATCHVER:=$(KERNEL_TESTING_PATCHVER)
 endif
 
-LINUX_VERSION-4.14 = .275
+LINUX_VERSION-4.14 = .285
 
-LINUX_KERNEL_HASH-4.14.275 = 100a9960fb2d8e079c9feeef640715a7fb749ed728a57e427f9e2443212e58f9
+LINUX_KERNEL_HASH-4.14.285 = dac9389b5cd46f0755403b17df0d0a4a28a7ab394f064a2bc30283ca0466d37a
 
 remove_uri_prefix=$(subst git://,,$(subst http://,,$(subst https://,,$(1))))
 sanitize_uri=$(call qstrip,$(subst @,_,$(subst :,_,$(subst .,_,$(subst -,_,$(subst /,_,$(1)))))))
-- 
2.34.1

