From f19081ccd0930b4948b8d77ac2dde708f173aece Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <michal.hrusecky@nic.cz>
Date: Thu, 2 Aug 2018 14:39:39 +0200
Subject: [PATCH] luajit: Try to build something on 64bits

---
 lang/luajit/Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/lang/luajit/Makefile b/lang/luajit/Makefile
index c1a52879a..c371c713e 100644
--- a/lang/luajit/Makefile
+++ b/lang/luajit/Makefile
@@ -37,7 +37,7 @@ TARGET_CFLAGS += $(FPIC) -std=gnu99
 HOST_CFLAGS += $(FPIC) -std=gnu99
 
 ifeq ($(HOST_ARCH),x86_64)
-  ifeq ($(CONFIG_x86_64),)
+  ifeq ($(CONFIG_ARCH_64BIT),)
     HOST_BITS := -m32
   endif
 endif
@@ -71,7 +71,7 @@ define Package/luajit/install
 	$(INSTALL_DIR) $(1)/usr/lib/
 	$(CP) $(PKG_INSTALL_DIR)/usr/lib/*.so* $(1)/usr/lib/
 	$(INSTALL_DIR) $(1)/usr/bin
-	$(CP) $(PKG_INSTALL_DIR)/usr/bin/luajit-2.1.0-beta2 $(1)/usr/bin/$(PKG_NAME)
+	$(CP) $(PKG_INSTALL_DIR)/usr/bin/luajit-2.1.0* $(1)/usr/bin/$(PKG_NAME)
 endef
 
 define Host/Compile
@@ -85,7 +85,7 @@ define Host/Install
 	$(MAKE) $(HOST_JOBS) -C $(HOST_BUILD_DIR) \
 		DPREFIX=$(STAGING_DIR_HOSTPKG) \
 		install
-	$(CP) $(STAGING_DIR_HOSTPKG)/bin/luajit-2.1.0-beta2 $(STAGING_DIR_HOSTPKG)/bin/$(PKG_NAME)
+	$(CP) $(STAGING_DIR_HOSTPKG)/bin/luajit* $(STAGING_DIR_HOSTPKG)/bin/$(PKG_NAME)
 endef
 
 $(eval $(call HostBuild,luajit))
-- 
2.24.0
