From d7dc3ff28088a97298f2f3f9a0168d5c3d10d382 Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <michal.hrusecky@nic.cz>
Date: Mon, 2 Feb 2019 15:53:28 +0100
Subject: [PATCH] python: disable weird dependencies

diff --git a/lang/python/pyodbc/Makefile b/lang/python/pyodbc/Makefile
index aec3e7e63..f5862e4ad 100644
--- a/lang/python/pyodbc/Makefile
+++ b/lang/python/pyodbc/Makefile
@@ -34,16 +34,16 @@ endef
 define Package/python-pyodbc
 $(call Package/python-pyodbc/Default)
   TITLE:=python-pyodbc
-  DEPENDS+=+PACKAGE_python-pyodbc:python-light \
-           +PACKAGE_python-pyodbc:python-logging \
-           +PACKAGE_python-pyodbc:python-openssl
+  DEPENDS+=+python-light \
+           +python-logging \
+           +python-openssl
   VARIANT:=python
 endef
 
 define Package/python3-pyodbc
 $(call Package/python-pyodbc/Default)
   TITLE:=python3-pyodbc
-  DEPENDS+=+PACKAGE_python3-pyodbc:python3-light
+  DEPENDS+=+python3-light
   VARIANT:=python3
 endef
 
diff --git a/lang/python/python-asn1crypto/Makefile b/lang/python/python-asn1crypto/Makefile
index 73e2f6305..2032dbcfa 100644
--- a/lang/python/python-asn1crypto/Makefile
+++ b/lang/python/python-asn1crypto/Makefile
@@ -36,14 +36,14 @@ endef
 define Package/python-asn1crypto
 $(call Package/python-asn1crypto/Default)
   TITLE:=python-asn1crypto
-  DEPENDS:=+PACKAGE_python-asn1crypto:python-light
+  DEPENDS:=+python-light
   VARIANT:=python
 endef
 
 define Package/python3-asn1crypto
 $(call Package/python-asn1crypto/Default)
   TITLE:=python3-asn1crypto
-  DEPENDS:=+PACKAGE_python3-asn1crypto:python3-light
+  DEPENDS:=+python3-light
   VARIANT:=python3
 endef
 
diff --git a/lang/python/python-attrs/Makefile b/lang/python/python-attrs/Makefile
index d76a78347..77e048a8d 100644
--- a/lang/python/python-attrs/Makefile
+++ b/lang/python/python-attrs/Makefile
@@ -37,14 +37,14 @@ endef
 define Package/python-attrs
 $(call Package/python-attrs/Default)
   TITLE:=Classes Without Boilerplate
-  DEPENDS:=+PACKAGE_python-attrs:python-light
+  DEPENDS:=+python-light
   VARIANT:=python
 endef
 
 define Package/python3-attrs
 $(call Package/python-attrs/Default)
   TITLE:=Classes Without Boilerplate
-  DEPENDS:=+PACKAGE_python3-attrs:python3-light
+  DEPENDS:=+python3-light
   VARIANT:=python3
 endef
 
diff --git a/lang/python/python-automat/Makefile b/lang/python/python-automat/Makefile
index 0c0ae5947..66fb25ea0 100644
--- a/lang/python/python-automat/Makefile
+++ b/lang/python/python-automat/Makefile
@@ -38,9 +38,9 @@ define Package/python-automat
 $(call Package/python-automat/Default)
   TITLE:=Concise, idiomatic finite-state automata
   DEPENDS:= \
-      +PACKAGE_python-automat:python-light \
-      +PACKAGE_python-automat:python-attrs \
-      +PACKAGE_python-automat:python-six
+      +python-light \
+      +python-attrs \
+      +python-six
   VARIANT:=python
 endef
 
@@ -48,9 +48,9 @@ define Package/python3-automat
 $(call Package/python-automat/Default)
   TITLE:=Concise, idiomatic finite-state automata
   DEPENDS:= \
-      +PACKAGE_python3-automat:python3-light \
-      +PACKAGE_python3-automat:python3-attrs \
-      +PACKAGE_python3-automat:python3-six
+      +python3-light \
+      +python3-attrs \
+      +python3-six
   VARIANT:=python3
 endef
 
diff --git a/lang/python/python-cffi/Makefile b/lang/python/python-cffi/Makefile
index 5e27adb13..093ed6e10 100644
--- a/lang/python/python-cffi/Makefile
+++ b/lang/python/python-cffi/Makefile
@@ -43,14 +43,14 @@ endef
 define Package/python-cffi
 $(call Package/python-cffi/Default)
   TITLE:=python-cffi
-  DEPENDS+=+PACKAGE_python-cffi:python-light +PACKAGE_python-cffi:python-pycparser
+  DEPENDS+=+python-light +python-pycparser
   VARIANT:=python
 endef
 
 define Package/python3-cffi
 $(call Package/python-cffi/Default)
   TITLE:=python3-cffi
-  DEPENDS+=+PACKAGE_python3-cffi:python3-light +PACKAGE_python3-cffi:python3-pycparser
+  DEPENDS+=+python3-light +python3-pycparser
   VARIANT:=python3
 endef
 
diff --git a/lang/python/python-constantly/Makefile b/lang/python/python-constantly/Makefile
index e1fa855ff..71fbbdee4 100644
--- a/lang/python/python-constantly/Makefile
+++ b/lang/python/python-constantly/Makefile
@@ -37,14 +37,14 @@ endef
 define Package/python-constantly
 $(call Package/python-constantly/Default)
   TITLE:=Symbolic constants in Python
-  DEPENDS:=+PACKAGE_python-constantly:python-light
+  DEPENDS:=+python-light
   VARIANT:=python
 endef
 
 define Package/python3-constantly
 $(call Package/python-constantly/Default)
   TITLE:=Symbolic constants in Python
-  DEPENDS:=+PACKAGE_python3-constantly:python3-light
+  DEPENDS:=+python3-light
   VARIANT:=python3
 endef
 
diff --git a/lang/python/python-crypto/Makefile b/lang/python/python-crypto/Makefile
index 551fd94fa..bc7b02413 100644
--- a/lang/python/python-crypto/Makefile
+++ b/lang/python/python-crypto/Makefile
@@ -37,14 +37,14 @@ endef
 define Package/python-crypto
 $(call Package/python-crypto/Default)
   TITLE:=python-crypto
-  DEPENDS:=+libgmp +PACKAGE_python-crypto:python
+  DEPENDS:=+libgmp +python
   VARIANT:=python
 endef
 
 define Package/python3-crypto
 $(call Package/python-crypto/Default)
   TITLE:=python3-crypto
-  DEPENDS:=+libgmp +PACKAGE_python3-crypto:python3
+  DEPENDS:=+libgmp +python3
   VARIANT:=python3
 endef
 
diff --git a/lang/python/python-cryptography/Makefile b/lang/python/python-cryptography/Makefile
index 02e95dabc..5a91d0e7b 100644
--- a/lang/python/python-cryptography/Makefile
+++ b/lang/python/python-cryptography/Makefile
@@ -39,13 +39,13 @@ define Package/python-cryptography
 $(call Package/python-cryptography/Default)
   TITLE:=python-cryptography
   DEPENDS+= \
-      +PACKAGE_python-cryptography:python \
-      +PACKAGE_python-cryptography:python-cffi \
-      +PACKAGE_python-cryptography:python-enum34 \
-      +PACKAGE_python-cryptography:python-idna \
-      +PACKAGE_python-cryptography:python-ipaddress \
-      +PACKAGE_python-cryptography:python-asn1crypto \
-      +PACKAGE_python-cryptography:python-six
+      +python \
+      +python-cffi \
+      +python-enum34 \
+      +python-idna \
+      +python-ipaddress \
+      +python-asn1crypto \
+      +python-six
   VARIANT:=python
 endef
 
@@ -53,11 +53,11 @@ define Package/python3-cryptography
 $(call Package/python-cryptography/Default)
   TITLE:=python3-cryptography
   DEPENDS+= \
-      +PACKAGE_python3-cryptography:python3 \
-      +PACKAGE_python3-cryptography:python3-cffi \
-      +PACKAGE_python3-cryptography:python3-idna \
-      +PACKAGE_python3-cryptography:python3-asn1crypto \
-      +PACKAGE_python3-cryptography:python3-six
+      +python3 \
+      +python3-cffi \
+      +python3-idna \
+      +python3-asn1crypto \
+      +python3-six
   VARIANT:=python3
 endef
 
diff --git a/lang/python/python-enum34/Makefile b/lang/python/python-enum34/Makefile
index 90b54c4c1..11b0576fc 100644
--- a/lang/python/python-enum34/Makefile
+++ b/lang/python/python-enum34/Makefile
@@ -36,7 +36,7 @@ endef
 define Package/python-enum34
 $(call Package/python-enum34/Default)
   TITLE:=python-enum34
-  DEPENDS:=+PACKAGE_python-enum34:python-light
+  DEPENDS:=+python-light
   VARIANT:=python
 endef
 
diff --git a/lang/python/python-evdev/Makefile b/lang/python/python-evdev/Makefile
index 4bdf5212f..80a29f1ab 100644
--- a/lang/python/python-evdev/Makefile
+++ b/lang/python/python-evdev/Makefile
@@ -38,8 +38,8 @@ define Package/python-evdev
 $(call Package/python-evdev/Default)
   TITLE:=python-evdev
   DEPENDS:= \
-     +PACKAGE_python-evdev:python-light \
-     +PACKAGE_python-evdev:python-ctypes
+     +python-light \
+     +python-ctypes
   VARIANT:=python
 endef
 
@@ -47,8 +47,8 @@ define Package/python3-evdev
 $(call Package/python-evdev/Default)
   TITLE:=python3-evdev
   DEPENDS:= \
-      +PACKAGE_python3-evdev:python3-light \
-      +PACKAGE_python3-evdev:python3-ctypes
+      +python3-light \
+      +python3-ctypes
   VARIANT:=python3
 endef
 
diff --git a/lang/python/python-gmpy2/Makefile b/lang/python/python-gmpy2/Makefile
index 991741365..336499af7 100644
--- a/lang/python/python-gmpy2/Makefile
+++ b/lang/python/python-gmpy2/Makefile
@@ -37,14 +37,14 @@ endef
 define Package/python-gmpy2
 $(call Package/python-gmpy2/Default)
   TITLE:=python-gmpy2
-  DEPENDS:=+libgmp +PACKAGE_python-gmpy2:python-light
+  DEPENDS:=+libgmp +python-light
   VARIANT:=python
 endef
 
 define Package/python3-gmpy2
 $(call Package/python-gmpy2/Default)
   TITLE:=python3-gmpy2
-  DEPENDS:=+libgmp +PACKAGE_python3-gmpy2:python3-light
+  DEPENDS:=+libgmp +python3-light
   VARIANT:=python3
 endef
 
diff --git a/lang/python/python-gnupg/Makefile b/lang/python/python-gnupg/Makefile
index 0c209a751..c84f3c96f 100644
--- a/lang/python/python-gnupg/Makefile
+++ b/lang/python/python-gnupg/Makefile
@@ -34,14 +34,14 @@ endef
 define Package/python-gnupg
 $(call Package/python-gnupg/Default)
   TITLE:=python-pyodbc
-  DEPENDS+=+PACKAGE_python-gnupg:python-light
+  DEPENDS+=+python-light
   VARIANT:=python
 endef
 
 define Package/python3-gnupg
 $(call Package/python-gnupg/Default)
   TITLE:=python3-gnupg
-  DEPENDS+=+PACKAGE_python3-gnupg:python3-light
+  DEPENDS+=+python3-light
   VARIANT:=python3
 endef
 
diff --git a/lang/python/python-hyperlink/Makefile b/lang/python/python-hyperlink/Makefile
index 36613abc1..8aee754be 100644
--- a/lang/python/python-hyperlink/Makefile
+++ b/lang/python/python-hyperlink/Makefile
@@ -38,8 +38,8 @@ define Package/python-hyperlink
 $(call Package/python-hyperlink/Default)
   TITLE:=Pure-Python immutable URLs
   DEPENDS:= \
-      +PACKAGE_python-hyperlink:python-light \
-      +PACKAGE_python-hyperlink:python-idna
+      +python-light \
+      +python-idna
   VARIANT:=python
 endef
 
@@ -47,8 +47,8 @@ define Package/python3-hyperlink
 $(call Package/python-hyperlink/Default)
   TITLE:=Pure-Python immutable URLs
   DEPENDS:= \
-      +PACKAGE_python3-hyperlink:python3-light \
-      +PACKAGE_python3-hyperlink:python3-idna
+      +python3-light \
+      +python3-idna
   VARIANT:=python3
 endef
 
diff --git a/lang/python/python-idna/Makefile b/lang/python/python-idna/Makefile
index 826565fae..ea95c7e73 100644
--- a/lang/python/python-idna/Makefile
+++ b/lang/python/python-idna/Makefile
@@ -37,14 +37,14 @@ endef
 define Package/python-idna
 $(call Package/python-idna/Default)
   TITLE:=python-idna
-  DEPENDS:=+PACKAGE_python-idna:python-light +PACKAGE_python-idna:python-codecs
+  DEPENDS:=+python-light +python-codecs
   VARIANT:=python
 endef
 
 define Package/python3-idna
 $(call Package/python-idna/Default)
   TITLE:=python3-idna
-  DEPENDS:=+PACKAGE_python3-idna:python3-light +PACKAGE_python3-idna:python3-codecs
+  DEPENDS:=+python3-light +python3-codecs
   VARIANT:=python3
 endef
 
diff --git a/lang/python/python-incremental/Makefile b/lang/python/python-incremental/Makefile
index 107cfaf3f..3afff6fe8 100644
--- a/lang/python/python-incremental/Makefile
+++ b/lang/python/python-incremental/Makefile
@@ -37,14 +37,14 @@ endef
 define Package/python-incremental
 $(call Package/python-incremental/Default)
   TITLE:=Versions your Python projects
-  DEPENDS:=+PACKAGE_python-incremental:python-light
+  DEPENDS:=+python-light
   VARIANT:=python
 endef
 
 define Package/python3-incremental
 $(call Package/python-incremental/Default)
   TITLE:=Versions your Python projects
-  DEPENDS:=+PACKAGE_python3-incremental:python3-light
+  DEPENDS:=+python3-light
   VARIANT:=python3
 endef
 
diff --git a/lang/python/python-ipaddress/Makefile b/lang/python/python-ipaddress/Makefile
index 333ea3ee3..03ea5c83d 100644
--- a/lang/python/python-ipaddress/Makefile
+++ b/lang/python/python-ipaddress/Makefile
@@ -35,7 +35,7 @@ endef
 define Package/python-ipaddress
 $(call Package/python-ipaddress/Default)
   TITLE:=Python 3.3+'s ipaddress
-  DEPENDS:=+PACKAGE_python-ipaddress:python-light
+  DEPENDS:=+python-light
   VARIANT:=python
 endef
 
diff --git a/lang/python/python-lxml/Makefile b/lang/python/python-lxml/Makefile
index 0c543fce4..6fd512b87 100644
--- a/lang/python/python-lxml/Makefile
+++ b/lang/python/python-lxml/Makefile
@@ -37,14 +37,14 @@ endef
 define Package/python-lxml
 $(call Package/python-lxml/Default)
   TITLE:=python-lxml
-  DEPENDS+=+PACKAGE_python-lxml:python-light +PACKAGE_python-lxml:python-codecs
+  DEPENDS+=+python-light +python-codecs
   VARIANT:=python
 endef
 
 define Package/python3-lxml
 $(call Package/python-lxml/Default)
   TITLE:=python3-lxml
-  DEPENDS+=+PACKAGE_python3-lxml:python3-light
+  DEPENDS+=+python3-light
   VARIANT:=python3
 endef
 
diff --git a/lang/python/python-parsley/Makefile b/lang/python/python-parsley/Makefile
index 53b50954c..37223a521 100644
--- a/lang/python/python-parsley/Makefile
+++ b/lang/python/python-parsley/Makefile
@@ -37,14 +37,14 @@ endef
 define Package/python-parsley
 $(call Package/python-parsley/Default)
   TITLE:=python-parsley
-  DEPENDS:=+PACKAGE_python-parsley:python-light
+  DEPENDS:=+python-light
   VARIANT:=python
 endef
 
 define Package/python3-parsley
 $(call Package/python-parsley/Default)
   TITLE:=python3-parsley
-  DEPENDS:=+PACKAGE_python3-parsley:python3-light
+  DEPENDS:=+python3-light
   VARIANT:=python3
 endef
 
diff --git a/lang/python/python-ply/Makefile b/lang/python/python-ply/Makefile
index f4c511730..56026b5c1 100644
--- a/lang/python/python-ply/Makefile
+++ b/lang/python/python-ply/Makefile
@@ -42,14 +42,14 @@ endef
 define Package/python-ply
 $(call Package/python-ply/Default)
   TITLE:=lex and yacc for Python
-  DEPENDS:=+PACKAGE_python-ply:python-light
+  DEPENDS:=+python-light
   VARIANT:=python
 endef
 
 define Package/python3-ply
 $(call Package/python-ply/Default)
   TITLE:=lex and yacc for Python
-  DEPENDS:=+PACKAGE_python3-ply:python3-light
+  DEPENDS:=+python3-light
   VARIANT:=python3
 endef
 
diff --git a/lang/python/python-pyasn1-modules/Makefile b/lang/python/python-pyasn1-modules/Makefile
index 7ad1e1f35..3048032f3 100644
--- a/lang/python/python-pyasn1-modules/Makefile
+++ b/lang/python/python-pyasn1-modules/Makefile
@@ -39,8 +39,8 @@ $(call Package/python-pyasn1-modules/Default)
   TITLE:=python-pyasn1-modules
   VARIANT:=python
   DEPENDS:= \
-      +PACKAGE_python-pyasn1-modules:python-light \
-      +PACKAGE_python-pyasn1-modules:python-pyasn1
+      +python-light \
+      +python-pyasn1
 endef
 
 define Package/python3-pyasn1-modules
@@ -48,8 +48,8 @@ $(call Package/python-pyasn1-modules/Default)
   TITLE:=python3-pyasn1-modules
   VARIANT:=python3
   DEPENDS:= \
-      +PACKAGE_python3-pyasn1-modules:python3-light \
-      +PACKAGE_python3-pyasn1-modules:python3-pyasn1
+      +python3-light \
+      +python3-pyasn1
 endef
 
 define Package/python-pyasn1-modules/description
diff --git a/lang/python/python-pyasn1/Makefile b/lang/python/python-pyasn1/Makefile
index cb97aabfe..6f9166c0b 100644
--- a/lang/python/python-pyasn1/Makefile
+++ b/lang/python/python-pyasn1/Makefile
@@ -37,14 +37,14 @@ endef
 define Package/python-pyasn1
 $(call Package/python-pyasn1/Default)
   TITLE:=ASN.1 library for Python
-  DEPENDS:=+PACKAGE_python-pyasn1:python-light
+  DEPENDS:=+python-light
   VARIANT:=python
 endef
 
 define Package/python3-pyasn1
 $(call Package/python-pyasn1/Default)
   TITLE:=ASN.1 library for Python
-  DEPENDS:=+PACKAGE_python3-pyasn1:python3-light
+  DEPENDS:=+python3-light
   VARIANT:=python3
 endef
 
diff --git a/lang/python/python-pycparser/Makefile b/lang/python/python-pycparser/Makefile
index 9a8b65cae..4f0b295ae 100644
--- a/lang/python/python-pycparser/Makefile
+++ b/lang/python/python-pycparser/Makefile
@@ -40,14 +40,14 @@ endef
 define Package/python-pycparser
 $(call Package/python-pycparser/Default)
   TITLE:=python-pycparser
-  DEPENDS:=+PACKAGE_python-pycparser:python-light +PACKAGE_python-pycparser:python-ply
+  DEPENDS:=+python-light +python-ply
   VARIANT:=python
 endef
 
 define Package/python3-pycparser
 $(call Package/python-pycparser/Default)
   TITLE:=python3-pycparser
-  DEPENDS:=+PACKAGE_python3-pycparser:python3-light +PACKAGE_python3-pycparser:python3-ply
+  DEPENDS:=+python3-light +python3-ply
   VARIANT:=python3
 endef
 
diff --git a/lang/python/python-pyopenssl/Makefile b/lang/python/python-pyopenssl/Makefile
index 3f8fc9058..3775b36e0 100644
--- a/lang/python/python-pyopenssl/Makefile
+++ b/lang/python/python-pyopenssl/Makefile
@@ -39,9 +39,9 @@ $(call Package/python-pyopenssl/Default)
   TITLE:=python-pyopenssl
   VARIANT:=python
   DEPENDS:= \
-      +PACKAGE_python-pyopenssl:python-light \
-      +PACKAGE_python-pyopenssl:python-cryptography \
-      +PACKAGE_python-pyopenssl:python-six
+      +python-light \
+      +python-cryptography \
+      +python-six
 endef
 
 define Package/python3-pyopenssl
@@ -49,9 +49,9 @@ $(call Package/python-pyopenssl/Default)
   TITLE:=python3-pyopenssl
   VARIANT:=python3
   DEPENDS:= \
-      +PACKAGE_python3-pyopenssl:python3-light \
-      +PACKAGE_python3-pyopenssl:python3-cryptography \
-      +PACKAGE_python3-pyopenssl:python3-six
+      +python3-light \
+      +python3-cryptography \
+      +python3-six
 endef
 
 define Package/python-pyopenssl/description
diff --git a/lang/python/python-pyptlib/Makefile b/lang/python/python-pyptlib/Makefile
index b003976a5..2a4f9c12e 100644
--- a/lang/python/python-pyptlib/Makefile
+++ b/lang/python/python-pyptlib/Makefile
@@ -36,7 +36,7 @@ endef
 define Package/python-pyptlib
 $(call Package/python-pyptlib/Default)
   TITLE:=python-pyptlib
-  DEPENDS:=+PACKAGE_python-pyptlib:python-light
+  DEPENDS:=+python-light
   VARIANT:=python
 endef
 
diff --git a/lang/python/python-service-identity/Makefile b/lang/python/python-service-identity/Makefile
index 552ba3d31..8175fe4a4 100644
--- a/lang/python/python-service-identity/Makefile
+++ b/lang/python/python-service-identity/Makefile
@@ -38,11 +38,11 @@ define Package/python-service-identity
 $(call Package/python-service-identity/Default)
   TITLE:=python-service-identity
   DEPENDS:= \
-      +PACKAGE_python-service-identity:python-light \
-      +PACKAGE_python-service-identity:python-attrs \
-      +PACKAGE_python-service-identity:python-pyasn1 \
-      +PACKAGE_python-service-identity:python-pyasn1-modules \
-      +PACKAGE_python-service-identity:python-pyopenssl
+      +python-light \
+      +python-attrs \
+      +python-pyasn1 \
+      +python-pyasn1-modules \
+      +python-pyopenssl
   VARIANT:=python
 endef
 
@@ -50,11 +50,11 @@ define Package/python3-service-identity
 $(call Package/python-service-identity/Default)
   TITLE:=python3-service-identity
   DEPENDS:= \
-      +PACKAGE_python3-service-identity:python3-light \
-      +PACKAGE_python3-service-identity:python3-attrs \
-      +PACKAGE_python3-service-identity:python3-pyasn1 \
-      +PACKAGE_python3-service-identity:python3-pyasn1-modules \
-      +PACKAGE_python3-service-identity:python3-pyopenssl
+      +python3-light \
+      +python3-attrs \
+      +python3-pyasn1 \
+      +python3-pyasn1-modules \
+      +python3-pyopenssl
   VARIANT:=python3
 endef
 
diff --git a/lang/python/python-six/Makefile b/lang/python/python-six/Makefile
index 2e7e4fa5b..9b99c1875 100644
--- a/lang/python/python-six/Makefile
+++ b/lang/python/python-six/Makefile
@@ -41,14 +41,14 @@ endef
 define Package/python-six
 $(call Package/python-six/Default)
   TITLE:=python-six
-  DEPENDS:=+PACKAGE_python-six:python-light
+  DEPENDS:=+python-light
   VARIANT:=python
 endef
 
 define Package/python3-six
 $(call Package/python-six/Default)
   TITLE:=python3-six
-  DEPENDS:=+PACKAGE_python3-six:python3-light
+  DEPENDS:=+python3-light
   VARIANT:=python3
 endef
 
diff --git a/lang/python/python-txsocksx/Makefile b/lang/python/python-txsocksx/Makefile
index e8a7605b1..3d6d45ee0 100644
--- a/lang/python/python-txsocksx/Makefile
+++ b/lang/python/python-txsocksx/Makefile
@@ -37,9 +37,9 @@ define Package/python-txsocksx
 $(call Package/python-txsocksx/Default)
   TITLE:=python-txsocksx
   DEPENDS:= \
-      +PACKAGE_python-txsocksx:python-light \
-      +PACKAGE_python-txsocksx:python-parsley \
-      +PACKAGE_python-txsocksx:twisted
+      +python-light \
+      +python-parsley \
+      +twisted
   VARIANT:=python
 endef
 
diff --git a/lang/python/python-yaml/Makefile b/lang/python/python-yaml/Makefile
index 7846ce267..7c793053d 100644
--- a/lang/python/python-yaml/Makefile
+++ b/lang/python/python-yaml/Makefile
@@ -37,14 +37,14 @@ endef
 define Package/python-yaml
 $(call Package/python-yaml/Default)
   TITLE:=YAML parser and emitter for Python
-  DEPENDS+=+PACKAGE_python-yaml:python-light +PACKAGE_python-yaml:python-codecs
+  DEPENDS+=+python-light +python-codecs
   VARIANT:=python
 endef
 
 define Package/python3-yaml
 $(call Package/python-yaml/Default)
   TITLE:=YAML parser and emitter for Python3
-  DEPENDS+=+PACKAGE_python3-yaml:python3-light
+  DEPENDS+=+python3-light
   VARIANT:=python3
 endef
 
diff --git a/lang/python/python-zope-interface/Makefile b/lang/python/python-zope-interface/Makefile
index 178f57805..4c407dca7 100644
--- a/lang/python/python-zope-interface/Makefile
+++ b/lang/python/python-zope-interface/Makefile
@@ -37,14 +37,14 @@ endef
 define Package/python-zope-interface
 $(call Package/python-zope-interface/Default)
   TITLE:=Interfaces for Python
-  DEPENDS:=+PACKAGE_python-zope-interface:python-light
+  DEPENDS:=+python-light
   VARIANT:=python
 endef
 
 define Package/python3-zope-interface
 $(call Package/python-zope-interface/Default)
   TITLE:=Interfaces for Python
-  DEPENDS:=+PACKAGE_python3-zope-interface:python3-light
+  DEPENDS:=+python3-light
   VARIANT:=python3
 endef
 
