From ae85b4dd5af766151ac145fa581602fd8fe5c50b Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Tue, 24 Sep 2019 20:29:10 +0200
Subject: [PATCH] postgresql: disable ASLR PIE

Signed-off-by: Josef Schlehofer <pepe.schlehofer@gmail.com>
---
 libs/postgresql/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/libs/postgresql/Makefile b/libs/postgresql/Makefile
index 810f59c55..9c9846dd7 100644
--- a/libs/postgresql/Makefile
+++ b/libs/postgresql/Makefile
@@ -8,6 +8,7 @@ PKG_NAME:=postgresql
 PKG_VERSION:=11.3
 PKG_RELEASE:=1
 PKG_MAINTAINER:=Daniel Golle <daniel@makrotopia.org>
+
 PKG_LICENSE:=PostgreSQL
 PKG_CPE_ID:=cpe:/a:postgresql:postgresql
 
@@ -24,6 +25,7 @@ PKG_FIXUP:=autoreconf
 PKG_MACRO_PATHS:=config
 PKG_BUILD_DEPENDS:=postgresql/host
 PKG_INSTALL:=1
+PKG_ASLR_PIE:=0
 
 include $(INCLUDE_DIR)/host-build.mk
 include $(INCLUDE_DIR)/package.mk
-- 
2.20.1

