From b3d1d6e3ea1aa5b3cb81c80e471fa26cd51560f6 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Wed, 25 Sep 2019 10:43:41 +0200
Subject: [PATCH] domoticz: disable PKG_ASLR_PIE

---
 utils/domoticz/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/utils/domoticz/Makefile b/utils/domoticz/Makefile
index e5cbe8c..1ecf97b 100644
--- a/utils/domoticz/Makefile
+++ b/utils/domoticz/Makefile
@@ -21,6 +21,7 @@ PKG_LICENSE_FILES:=License.txt
 PKG_BUILD_DEPENDS:=python3 minizip cereal
 PKG_BUILD_PARALLEL:=1
 PKG_USE_MIPS16:=0
+PKG_ASLR_PIE:=0
 
 CMAKE_INSTALL:=1
 
-- 
2.27.0

