From f17cb6ec4ab7b925f7392fc46d3cdbbe58010d9c Mon Sep 17 00:00:00 2001
From: Richard Muzik <richard.muzik@nic.cz>
Date: Thu, 27 Feb 2025 13:14:41 +0100
Subject: [PATCH] sunwait: add -fPIC compilation flag

---
 utils/sunwait/Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/utils/sunwait/Makefile b/utils/sunwait/Makefile
index 99d2a88a4..575fd1ee8 100644
--- a/utils/sunwait/Makefile
+++ b/utils/sunwait/Makefile
@@ -32,9 +32,9 @@ define Package/sunwait/description
 endef
 
 define Build/Compile
-	$(TARGET_CC) $(TARGET_CPPFLAGS) -o $(PKG_BUILD_DIR)/sunwait.o -c $(PKG_BUILD_DIR)/sunwait.c
-	$(TARGET_CC) $(TARGET_CPPFLAGS) -o $(PKG_BUILD_DIR)/sunriset.o -c $(PKG_BUILD_DIR)/sunriset.c
-	$(TARGET_CC) $(TARGET_CPPFLAGS) -o $(PKG_BUILD_DIR)/print.o -c $(PKG_BUILD_DIR)/print.c
+	$(TARGET_CC) $(TARGET_CPPFLAGS) -fPIC -o $(PKG_BUILD_DIR)/sunwait.o -c $(PKG_BUILD_DIR)/sunwait.c
+	$(TARGET_CC) $(TARGET_CPPFLAGS) -fPIC -o $(PKG_BUILD_DIR)/sunriset.o -c $(PKG_BUILD_DIR)/sunriset.c
+	$(TARGET_CC) $(TARGET_CPPFLAGS) -fPIC -o $(PKG_BUILD_DIR)/print.o -c $(PKG_BUILD_DIR)/print.c
 	$(TARGET_CC) $(TARGET_LDFLAGS) -o $(PKG_BUILD_DIR)/$1 $(PKG_BUILD_DIR)/sunwait.o $(PKG_BUILD_DIR)/sunriset.o $(PKG_BUILD_DIR)/print.o -lm
 endef
 
-- 
2.48.1

