From 85b698b52ede0e73c489dbcebba4fca00106903b Mon Sep 17 00:00:00 2001
From: Jan Pavlinec <jan.pavlinec@nic.cz>
Date: Fri, 8 Feb 2019 20:12:15 +0100
Subject: [PATCH] collectd: disable nut for now

---
 utils/collectd/Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/utils/collectd/Makefile b/utils/collectd/Makefile
index 70013ff..62f1897 100644
--- a/utils/collectd/Makefile
+++ b/utils/collectd/Makefile
@@ -21,7 +21,7 @@ PKG_REMOVE_FILES:=aclocal.m4 libltdl/aclocal.m4
 PKG_MAINTAINER:=Jo-Philipp Wich <jo@mein.io>, Hannu Nyman <hannu.nyman@iki.fi>
 
 PKG_INSTALL:=1
-PKG_BUILD_PARALLEL:=1
+PKG_BUILD_PARALLEL:=0
 PKG_USE_MIPS16:=0
 
 COLLECTD_PLUGINS_DISABLED:= \
@@ -72,6 +72,7 @@ COLLECTD_PLUGINS_DISABLED:= \
 	notify_email \
 	notify_nagios \
 	numa \
+	nut \
 	openldap \
 	openvz \
 	oracle \
@@ -151,7 +152,6 @@ COLLECTD_PLUGINS_SELECTED:= \
 	network \
 	nginx \
 	ntpd \
-	nut \
 	olsrd \
 	onewire \
 	openvpn \
@@ -372,7 +372,7 @@ $(eval $(call BuildPlugin,netlink,netlink input,netlink,+PACKAGE_collectd-mod-ne
 $(eval $(call BuildPlugin,network,network input/output,network,+PACKAGE_COLLECTD_ENCRYPTED_NETWORK:libgcrypt))
 $(eval $(call BuildPlugin,nginx,nginx status input,nginx,+PACKAGE_collectd-mod-nginx:libcurl))
 $(eval $(call BuildPlugin,ntpd,NTP daemon status input,ntpd,))
-$(eval $(call BuildPlugin,nut,UPS monitoring input,nut,+PACKAGE_collectd-mod-nut:nut-common))
+#$(eval $(call BuildPlugin,nut,UPS monitoring input,nut,+PACKAGE_collectd-mod-nut:nut-common))
 $(eval $(call BuildPlugin,olsrd,OLSRd status input,olsrd,))
 $(eval $(call BuildPlugin,onewire,onewire sensor input,onewire,+PACKAGE_collectd-mod-onewire:libow-capi @BROKEN))
 $(eval $(call BuildPlugin,openvpn,OpenVPN traffic/compression input,openvpn,))
-- 
2.7.4

