From 8c6a14a8e862059f7274a0dfa0e81228f36c5501 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Wed, 25 Sep 2019 10:43:41 +0200
Subject: [PATCH] domoticz: disable PKG_ASLR_PIE

---
 utils/domoticz/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/utils/domoticz/Makefile b/utils/domoticz/Makefile
index 7bd2ae06b..f31ac5e82 100644
--- a/utils/domoticz/Makefile
+++ b/utils/domoticz/Makefile
@@ -30,6 +30,7 @@ PKG_LICENSE_FILES:=License.txt
 PKG_BUILD_DEPENDS:=python3
 PKG_BUILD_PARALLEL:=1
 PKG_USE_MIPS16:=0
+PKG_ASLR_PIE:=0
 
 CMAKE_INSTALL:=1
 
-- 
2.20.1

