From e8aa3faa5232a4b69ae39fb1d3cabe2fe340ec18 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Sun, 20 Oct 2019 13:58:59 +0200
Subject: [PATCH] golang: disable ASLR PIE

Signed-off-by: Josef Schlehofer <pepe.schlehofer@gmail.com>
---
 lang/golang/golang/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lang/golang/golang/Makefile b/lang/golang/golang/Makefile
index 41837ac4d..3662419ef 100644
--- a/lang/golang/golang/Makefile
+++ b/lang/golang/golang/Makefile
@@ -28,6 +28,7 @@ PKG_BUILD_DEPENDS:=golang/host
 PKG_BUILD_DIR:=$(BUILD_DIR)/go-$(PKG_VERSION)
 PKG_BUILD_PARALLEL:=1
 PKG_USE_MIPS16:=0
+PKG_ASLR_PIE:=0
 
 PKG_GO_WORK_DIR:=$(PKG_BUILD_DIR)/.go_work
 PKG_GO_HOST_CACHE_DIR:=$(PKG_GO_WORK_DIR)/host_cache
-- 
2.20.1

